<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_45) on Fri Jan 10 12:31:19 CET 2014 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>JBioWHGraph (jbiowh-persistence 6.0.1 API)</title>
<meta name="date" content="2014-01-10">
<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="JBioWHGraph (jbiowh-persistence 6.0.1 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/JBioWHGraph.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../index-all.html">Index</a></li>
<li><a href="../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev Class</li>
<li><a href="../../../../org/jbiowhpersistence/utils/graph/JGraphXAdapter.html" title="class in org.jbiowhpersistence.utils.graph"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../index.html?org/jbiowhpersistence/utils/graph/JBioWHGraph.html" target="_top">Frames</a></li>
<li><a href="JBioWHGraph.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#field_summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field_detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<!-- ======== START OF CLASS DATA ======== -->
<div class="header">
<div class="subTitle">org.jbiowhpersistence.utils.graph</div>
<h2 title="Class JBioWHGraph" class="title">Class JBioWHGraph</h2>
</div>
<div class="contentContainer">
<ul class="inheritance">
<li><a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">java.lang.Object</a></li>
<li>
<ul class="inheritance">
<li>org.jbiowhpersistence.utils.graph.JBioWHGraph</li>
</ul>
</li>
</ul>
<div class="description">
<ul class="blockList">
<li class="blockList">
<dl>
<dt>Direct Known Subclasses:</dt>
<dd><a href="../../../../org/jbiowhpersistence/datasets/drug/drugbank/util/DrugPathwayGraph.html" title="class in org.jbiowhpersistence.datasets.drug.drugbank.util">DrugPathwayGraph</a>, <a href="../../../../org/jbiowhpersistence/datasets/protein/util/ProteinProteinGraph.html" title="class in org.jbiowhpersistence.datasets.protein.util">ProteinProteinGraph</a>, <a href="../../../../org/jbiowhpersistence/datasets/taxonomy/util/TaxonomyGraph.html" title="class in org.jbiowhpersistence.datasets.taxonomy.util">TaxonomyGraph</a></dd>
</dl>
<hr>
<br>
<pre>public class <span class="strong">JBioWHGraph</span>
extends <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a></pre>
<div class="block">This class creates a graph from any JBioWH main entity

 $Author: r78v10a07@gmail.com $ $LastChangedDate: 2013-03-27 15:25:50 +0100
 (Wed, 27 Mar 2013) $ $LastChangedRevision: 795 $</div>
<dl><dt><span class="strong">Since:</span></dt>
  <dd>Mar 25, 2013</dd></dl>
</li>
</ul>
</div>
<div class="summary">
<ul class="blockList">
<li class="blockList">
<!-- =========== FIELD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="field_summary">
<!--   -->
</a>
<h3>Field Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Field Summary table, listing fields, and an explanation">
<caption><span>Fields</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Field and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected int</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#depth">depth</a></strong></code>
<div class="block">The recursivity depth limit</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected org.jgrapht.ListenableGraph</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#graph">graph</a></strong></code>
<div class="block">The graph object</div>
</td>
</tr>
</table>
</li>
</ul>
<!-- ======== CONSTRUCTOR SUMMARY ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_summary">
<!--   -->
</a>
<h3>Constructor Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Constructor Summary table, listing constructors, and an explanation">
<caption><span>Constructors</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tr class="altColor">
<td class="colOne"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#JBioWHGraph()">JBioWHGraph</a></strong>()</code>
<div class="block">Creates a JBioWHGraph

 The depth is set to 0 and the Graph is created as an undirected graph</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#JBioWHGraph(int)">JBioWHGraph</a></strong>(int&nbsp;depth)</code>
<div class="block">Creates a JBioWHGraph

 The Graph is created as an undirected graph</div>
</td>
</tr>
<tr class="altColor">
<td class="colOne"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#JBioWHGraph(int, org.jgrapht.ListenableGraph)">JBioWHGraph</a></strong>(int&nbsp;depth,
           org.jgrapht.ListenableGraph&nbsp;graph)</code>
<div class="block">Creates a JBioWHGraph</div>
</td>
</tr>
<tr class="rowColor">
<td class="colOne"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#JBioWHGraph(org.jgrapht.ListenableGraph)">JBioWHGraph</a></strong>(org.jgrapht.ListenableGraph&nbsp;graph)</code>
<div class="block">Creates a JBioWHGraph</div>
</td>
</tr>
</table>
</li>
</ul>
<!-- ========== METHOD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="method_summary">
<!--   -->
</a>
<h3>Method Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Method Summary table, listing methods, and an explanation">
<caption><span>Methods</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#addBioWHNewVertex(java.lang.Object, java.lang.Object, int)">addBioWHNewVertex</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;source,
                 <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;dest,
                 int&nbsp;varDepth)</code>
<div class="block">Add the vertex dest and create the edge with the vertex source.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#addBioWHNewVertex(java.lang.Object, java.lang.Object, int, java.lang.Object)">addBioWHNewVertex</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;source,
                 <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;dest,
                 int&nbsp;varDepth,
                 <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;weightObject)</code>
<div class="block">Add the vertex dest and create the edge with the vertex source.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#containsVertex(java.lang.Object)">containsVertex</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;obj)</code>
<div class="block">Return true if the graph contains the object as vertex</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#edgeCount()">edgeCount</a></strong>()</code>
<div class="block">Count the graph edges</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#edgeToString(java.lang.Object, java.lang.Object, int)">edgeToString</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;vertex1,
            <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;vertex2,
            int&nbsp;varDepth)</code>
<div class="block">Return a string with the edge object in a format with tab in front</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>org.jgrapht.ListenableGraph</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#getGraph()">getGraph</a></strong>()</code>
<div class="block">Get the T tree in a graph object</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html" title="class in org.jbiowhpersistence.utils.graph">JBioWHGraph</a></code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#getGraphDijkstraShortestPath(java.lang.Object, java.lang.Object)">getGraphDijkstraShortestPath</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;vertex1,
                            <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;vertex2)</code>
<div class="block">Convenience method to find the shortest path via a single static method
 call.</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#getPathInList()">getPathInList</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#getPathInList(java.util.Collection)">getPathInList</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;org.jgrapht.graph.DefaultEdge&gt;&nbsp;edges)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#isGraphConnected()">isGraphConnected</a></strong>()</code>
<div class="block">Return true if the graph is connected</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.drug.drugbank.entities.DrugBank, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/DrugBank.html" title="class in org.jbiowhpersistence.datasets.drug.drugbank.entities">DrugBank</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from DrugBank entity</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.gene.genebank.entities.GeneBankCDS, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/gene/genebank/entities/GeneBankCDS.html" title="class in org.jbiowhpersistence.datasets.gene.genebank.entities">GeneBankCDS</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from GeneBankCDS entity</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.gene.genebank.entities.GeneBank, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/gene/genebank/entities/GeneBank.html" title="class in org.jbiowhpersistence.datasets.gene.genebank.entities">GeneBank</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from GeneBank entity</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from GeneInfo entity</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.gene.genome.entities.GenePTT, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/gene/genome/entities/GenePTT.html" title="class in org.jbiowhpersistence.datasets.gene.genome.entities">GenePTT</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from GenePTT entity</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.compound.KEGGCompound, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/compound/KEGGCompound.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.compound">KEGGCompound</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from KEGGCompound entity</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.enzyme.KEGGEnzyme, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/enzyme/KEGGEnzyme.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.enzyme">KEGGEnzyme</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from KEGGEnzyme entity</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.gene.KEGGGene, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/gene/KEGGGene.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.gene">KEGGGene</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from KEGGGene entity</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.glycan.KEGGGlycan, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/glycan/KEGGGlycan.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.glycan">KEGGGlycan</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from KEGGGlycan entity</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway.KEGGPathway, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/KEGGPathway.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway">KEGGPathway</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from KEGGPathway entity</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.reaction.KEGGReaction, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/reaction/KEGGReaction.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.reaction">KEGGReaction</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from KEGGReaction entity</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.ppi.entities.MIFEntryInteraction, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/ppi/entities/MIFEntryInteraction.html" title="class in org.jbiowhpersistence.datasets.ppi.entities">MIFEntryInteraction</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from MIFEntryInteraction
 entity</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.ppi.entities.MIFEntrySetEntry, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/ppi/entities/MIFEntrySetEntry.html" title="class in org.jbiowhpersistence.datasets.ppi.entities">MIFEntrySetEntry</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from MIFEntrySetEntry entity</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.ppi.entities.MIFEntrySet, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/ppi/entities/MIFEntrySet.html" title="class in org.jbiowhpersistence.datasets.ppi.entities">MIFEntrySet</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from MIFEntrySet entity</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(java.lang.Object)">populate</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;o)</code>
<div class="block">Populates the graph from the JBioWH main entity in the object o</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(java.lang.Object, int)">populate</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from the JBioWH main entity
 in the object o</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.disease.omim.entities.OMIM, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/disease/omim/entities/OMIM.html" title="class in org.jbiowhpersistence.datasets.disease.omim.entities">OMIM</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from OMIM entity</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.domain.pfam.entities.PfamAbioWH, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/domain/pfam/entities/PfamAbioWH.html" title="class in org.jbiowhpersistence.datasets.domain.pfam.entities">PfamAbioWH</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from PfamAbioWH entity</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.protein.entities.Protein, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from Protein entity</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.protclust.entities.UniRefEntry, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/protclust/entities/UniRefEntry.html" title="class in org.jbiowhpersistence.datasets.protclust.entities">UniRefEntry</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from UniRefEntry entity</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.protclust.entities.UniRefMember, int)">populate</a></strong>(<a href="../../../../org/jbiowhpersistence/datasets/protclust/entities/UniRefMember.html" title="class in org.jbiowhpersistence.datasets.protclust.entities">UniRefMember</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from UniRefMember entity</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populateCollection(java.util.Collection, java.lang.Object, int)">populateCollection</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&nbsp;col,
                  <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;o,
                  int&nbsp;varDepth)</code>
<div class="block">Add each element of the collection as a new vertex connected to the
 object o</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populateCollection(java.util.Collection, java.lang.Object, int, java.lang.Object)">populateCollection</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&nbsp;col,
                  <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;o,
                  int&nbsp;varDepth,
                  <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;weightObject)</code>
<div class="block">Add each element of the collection as a new vertex connected to the
 object o in a weighted graph</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>protected <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#tabToString(int)">tabToString</a></strong>(int&nbsp;varDepth)</code>
<div class="block">Return a string with a number of tabs equal to varDepth</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#vertexCount()">vertexCount</a></strong>()</code>
<div class="block">Count the graph vertex</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a></code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#vertexSet()">vertexSet</a></strong>()</code>
<div class="block">Get the graph vertex in a collection object</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a></code></td>
<td class="colLast"><code><strong><a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#vertexToString(java.lang.Object, int)">vertexToString</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;vertex,
              int&nbsp;varDepth)</code>
<div class="block">Return a string with the vertex object in a format with tab in front</div>
</td>
</tr>
</table>
<ul class="blockList">
<li class="blockList"><a name="methods_inherited_from_class_java.lang.Object">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;java.lang.<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a></h3>
<code><a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true#clone()" title="class or interface in java.lang">clone</a>, <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true#equals(java.lang.Object)" title="class or interface in java.lang">equals</a>, <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true#finalize()" title="class or interface in java.lang">finalize</a>, <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true#getClass()" title="class or interface in java.lang">getClass</a>, <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true#hashCode()" title="class or interface in java.lang">hashCode</a>, <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true#notify()" title="class or interface in java.lang">notify</a>, <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true#notifyAll()" title="class or interface in java.lang">notifyAll</a>, <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true#toString()" title="class or interface in java.lang">toString</a>, <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true#wait()" title="class or interface in java.lang">wait</a>, <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true#wait(long)" title="class or interface in java.lang">wait</a>, <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true#wait(long, int)" title="class or interface in java.lang">wait</a></code></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="details">
<ul class="blockList">
<li class="blockList">
<!-- ============ FIELD DETAIL =========== -->
<ul class="blockList">
<li class="blockList"><a name="field_detail">
<!--   -->
</a>
<h3>Field Detail</h3>
<a name="depth">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>depth</h4>
<pre>protected&nbsp;int depth</pre>
<div class="block">The recursivity depth limit</div>
</li>
</ul>
<a name="graph">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>graph</h4>
<pre>protected&nbsp;org.jgrapht.ListenableGraph graph</pre>
<div class="block">The graph object</div>
</li>
</ul>
</li>
</ul>
<!-- ========= CONSTRUCTOR DETAIL ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_detail">
<!--   -->
</a>
<h3>Constructor Detail</h3>
<a name="JBioWHGraph()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>JBioWHGraph</h4>
<pre>public&nbsp;JBioWHGraph()</pre>
<div class="block">Creates a JBioWHGraph

 The depth is set to 0 and the Graph is created as an undirected graph</div>
</li>
</ul>
<a name="JBioWHGraph(org.jgrapht.ListenableGraph)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>JBioWHGraph</h4>
<pre>public&nbsp;JBioWHGraph(org.jgrapht.ListenableGraph&nbsp;graph)</pre>
<div class="block">Creates a JBioWHGraph</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>graph</code> - the graph to be handled</dd></dl>
</li>
</ul>
<a name="JBioWHGraph(int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>JBioWHGraph</h4>
<pre>public&nbsp;JBioWHGraph(int&nbsp;depth)</pre>
<div class="block">Creates a JBioWHGraph

 The Graph is created as an undirected graph</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>depth</code> - the recursivity depth limit</dd></dl>
</li>
</ul>
<a name="JBioWHGraph(int, org.jgrapht.ListenableGraph)">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>JBioWHGraph</h4>
<pre>public&nbsp;JBioWHGraph(int&nbsp;depth,
           org.jgrapht.ListenableGraph&nbsp;graph)</pre>
<div class="block">Creates a JBioWHGraph</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>depth</code> - the recursivity depth limit</dd><dd><code>graph</code> - the graph to be handled</dd></dl>
</li>
</ul>
</li>
</ul>
<!-- ============ METHOD DETAIL ========== -->
<ul class="blockList">
<li class="blockList"><a name="method_detail">
<!--   -->
</a>
<h3>Method Detail</h3>
<a name="getGraph()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGraph</h4>
<pre>public&nbsp;org.jgrapht.ListenableGraph&nbsp;getGraph()</pre>
<div class="block">Get the T tree in a graph object</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the T tree in a graph object</dd></dl>
</li>
</ul>
<a name="vertexSet()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>vertexSet</h4>
<pre>public&nbsp;<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&nbsp;vertexSet()</pre>
<div class="block">Get the graph vertex in a collection object</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the graph vertex in a collection object</dd></dl>
</li>
</ul>
<a name="vertexCount()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>vertexCount</h4>
<pre>public&nbsp;int&nbsp;vertexCount()</pre>
<div class="block">Count the graph vertex</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the graph vertex number</dd></dl>
</li>
</ul>
<a name="edgeCount()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>edgeCount</h4>
<pre>public&nbsp;int&nbsp;edgeCount()</pre>
<div class="block">Count the graph edges</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the graph edges number</dd></dl>
</li>
</ul>
<a name="containsVertex(java.lang.Object)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>containsVertex</h4>
<pre>public&nbsp;boolean&nbsp;containsVertex(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;obj)</pre>
<div class="block">Return true if the graph contains the object as vertex</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>obj</code> - the object to be checked as a graph vertex</dd>
<dt><span class="strong">Returns:</span></dt><dd>true if the graph contains the object as vertex</dd></dl>
</li>
</ul>
<a name="isGraphConnected()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>isGraphConnected</h4>
<pre>public&nbsp;boolean&nbsp;isGraphConnected()</pre>
<div class="block">Return true if the graph is connected</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>true if the graph is connected</dd></dl>
</li>
</ul>
<a name="getGraphDijkstraShortestPath(java.lang.Object, java.lang.Object)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGraphDijkstraShortestPath</h4>
<pre>public&nbsp;<a href="../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html" title="class in org.jbiowhpersistence.utils.graph">JBioWHGraph</a>&nbsp;getGraphDijkstraShortestPath(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;vertex1,
                                       <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;vertex2)</pre>
<div class="block">Convenience method to find the shortest path via a single static method
 call. If you need a more advanced search (e.g. limited by radius, or
 computation of the path length), use the constructor instead.</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>vertex1</code> - the vertex at which the path should start</dd><dd><code>vertex2</code> - the vertex at which the path should end</dd>
<dt><span class="strong">Returns:</span></dt><dd>a JBioWHGraph with the path found</dd></dl>
</li>
</ul>
<a name="populate(java.lang.Object)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>public&nbsp;void&nbsp;populate(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;o)</pre>
<div class="block">Populates the graph from the JBioWH main entity in the object o</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the JBioWH main entity to start the graph</dd></dl>
</li>
</ul>
<a name="vertexToString(java.lang.Object, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>vertexToString</h4>
<pre>protected&nbsp;<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;vertexToString(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;vertex,
                    int&nbsp;varDepth)</pre>
<div class="block">Return a string with the vertex object in a format with tab in front</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>vertex</code> - the vertex name</dd><dd><code>varDepth</code> - the vertex depth</dd>
<dt><span class="strong">Returns:</span></dt><dd>a string with the vertex object in a format with tab in front</dd></dl>
</li>
</ul>
<a name="edgeToString(java.lang.Object, java.lang.Object, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>edgeToString</h4>
<pre>protected&nbsp;<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;edgeToString(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;vertex1,
                  <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;vertex2,
                  int&nbsp;varDepth)</pre>
<div class="block">Return a string with the edge object in a format with tab in front</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>vertex1</code> - the first vertex name</dd><dd><code>vertex2</code> - the second vertex name</dd><dd><code>varDepth</code> - the vertex depth</dd>
<dt><span class="strong">Returns:</span></dt><dd>a string with the edge object in a format with tab in front</dd></dl>
</li>
</ul>
<a name="populate(java.lang.Object, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from the JBioWH main entity
 in the object o</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the JBioWH main entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.domain.pfam.entities.PfamAbioWH, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/domain/pfam/entities/PfamAbioWH.html" title="class in org.jbiowhpersistence.datasets.domain.pfam.entities">PfamAbioWH</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from PfamAbioWH entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the PfamAbioWH entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.disease.omim.entities.OMIM, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/disease/omim/entities/OMIM.html" title="class in org.jbiowhpersistence.datasets.disease.omim.entities">OMIM</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from OMIM entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the OMIM entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.protclust.entities.UniRefMember, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/protclust/entities/UniRefMember.html" title="class in org.jbiowhpersistence.datasets.protclust.entities">UniRefMember</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from UniRefMember entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the UniRefMember entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.protclust.entities.UniRefEntry, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/protclust/entities/UniRefEntry.html" title="class in org.jbiowhpersistence.datasets.protclust.entities">UniRefEntry</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from UniRefEntry entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the UniRefEntry entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.reaction.KEGGReaction, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/reaction/KEGGReaction.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.reaction">KEGGReaction</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from KEGGReaction entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the KEGGReaction entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.glycan.KEGGGlycan, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/glycan/KEGGGlycan.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.glycan">KEGGGlycan</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from KEGGGlycan entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the KEGGGlycan entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.gene.KEGGGene, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/gene/KEGGGene.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.gene">KEGGGene</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from KEGGGene entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the KEGGGene entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.enzyme.KEGGEnzyme, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/enzyme/KEGGEnzyme.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.enzyme">KEGGEnzyme</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from KEGGEnzyme entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the KEGGEnzyme entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.compound.KEGGCompound, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/compound/KEGGCompound.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.compound">KEGGCompound</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from KEGGCompound entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the KEGGCompound entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.ppi.entities.MIFEntryInteraction, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/ppi/entities/MIFEntryInteraction.html" title="class in org.jbiowhpersistence.datasets.ppi.entities">MIFEntryInteraction</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from MIFEntryInteraction
 entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the MIFEntryInteraction entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.ppi.entities.MIFEntrySetEntry, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/ppi/entities/MIFEntrySetEntry.html" title="class in org.jbiowhpersistence.datasets.ppi.entities">MIFEntrySetEntry</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from MIFEntrySetEntry entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the MIFEntrySetEntry entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.ppi.entities.MIFEntrySet, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/ppi/entities/MIFEntrySet.html" title="class in org.jbiowhpersistence.datasets.ppi.entities">MIFEntrySet</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from MIFEntrySet entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the MIFEntrySet entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.gene.genome.entities.GenePTT, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/gene/genome/entities/GenePTT.html" title="class in org.jbiowhpersistence.datasets.gene.genome.entities">GenePTT</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from GenePTT entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the GenePTT entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from GeneInfo entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the GeneInfo entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.gene.genebank.entities.GeneBank, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/gene/genebank/entities/GeneBank.html" title="class in org.jbiowhpersistence.datasets.gene.genebank.entities">GeneBank</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from GeneBank entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the GenePTT entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.gene.genebank.entities.GeneBankCDS, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/gene/genebank/entities/GeneBankCDS.html" title="class in org.jbiowhpersistence.datasets.gene.genebank.entities">GeneBankCDS</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from GeneBankCDS entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the GenePTT entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.protein.entities.Protein, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from Protein entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the Protein entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.drug.drugbank.entities.DrugBank, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/DrugBank.html" title="class in org.jbiowhpersistence.datasets.drug.drugbank.entities">DrugBank</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from DrugBank entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the DrugBank entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populate(org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway.KEGGPathway, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/KEGGPathway.html" title="class in org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway">KEGGPathway</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from KEGGPathway entity</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the KEGGPathway entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populateCollection(java.util.Collection, java.lang.Object, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populateCollection</h4>
<pre>protected&nbsp;void&nbsp;populateCollection(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&nbsp;col,
                      <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;o,
                      int&nbsp;varDepth)</pre>
<div class="block">Add each element of the collection as a new vertex connected to the
 object o</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>col</code> - collection with the JBioWH entities to be added as new
 vertexes</dd><dd><code>o</code> - JBioWH entity which is a vertex that is connected to all
 elements in the collection</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="populateCollection(java.util.Collection, java.lang.Object, int, java.lang.Object)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>populateCollection</h4>
<pre>protected&nbsp;void&nbsp;populateCollection(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&nbsp;col,
                      <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;o,
                      int&nbsp;varDepth,
                      <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;weightObject)</pre>
<div class="block">Add each element of the collection as a new vertex connected to the
 object o in a weighted graph</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>col</code> - collection with the JBioWH entities to be added as new
 vertexes</dd><dd><code>o</code> - JBioWH entity which is a vertex that is connected to all
 elements in the collection</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd><dd><code>weightObject</code> - the object to increment the edge weight</dd></dl>
</li>
</ul>
<a name="tabToString(int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>tabToString</h4>
<pre>protected&nbsp;<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;tabToString(int&nbsp;varDepth)</pre>
<div class="block">Return a string with a number of tabs equal to varDepth</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>varDepth</code> - the recursive depth for parent level</dd>
<dt><span class="strong">Returns:</span></dt><dd>a string with a number of tabs equal to varDepth</dd></dl>
</li>
</ul>
<a name="addBioWHNewVertex(java.lang.Object, java.lang.Object, int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>addBioWHNewVertex</h4>
<pre>protected&nbsp;void&nbsp;addBioWHNewVertex(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;source,
                     <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;dest,
                     int&nbsp;varDepth)</pre>
<div class="block">Add the vertex dest and create the edge with the vertex source. If the
 vertex already exist then just the edge is created</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>source</code> - source vertex to be connected with the dest vertex</dd><dd><code>dest</code> - destination vertex to be connected with the source vertex</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
<a name="addBioWHNewVertex(java.lang.Object, java.lang.Object, int, java.lang.Object)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>addBioWHNewVertex</h4>
<pre>protected&nbsp;void&nbsp;addBioWHNewVertex(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;source,
                     <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;dest,
                     int&nbsp;varDepth,
                     <a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&nbsp;weightObject)</pre>
<div class="block">Add the vertex dest and create the edge with the vertex source. If the
 vertex already exist then just the edge is created.

 This method verify if the edge is created then increment the weight of
 this edge</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>source</code> - source vertex to be connected with the dest vertex</dd><dd><code>dest</code> - destination vertex to be connected with the source vertex</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd><dd><code>weightObject</code> - the MIF25 interaction that report the protein-protein
 interaction</dd></dl>
</li>
</ul>
<a name="getPathInList()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getPathInList</h4>
<pre>public&nbsp;<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&gt;&nbsp;getPathInList()</pre>
</li>
</ul>
<a name="getPathInList(java.util.Collection)">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>getPathInList</h4>
<pre>public&nbsp;<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a>&gt;&nbsp;getPathInList(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;org.jgrapht.graph.DefaultEdge&gt;&nbsp;edges)</pre>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- ========= END OF CLASS DATA ========= -->
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/JBioWHGraph.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../index-all.html">Index</a></li>
<li><a href="../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev Class</li>
<li><a href="../../../../org/jbiowhpersistence/utils/graph/JGraphXAdapter.html" title="class in org.jbiowhpersistence.utils.graph"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../index.html?org/jbiowhpersistence/utils/graph/JBioWHGraph.html" target="_top">Frames</a></li>
<li><a href="JBioWHGraph.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#field_summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field_detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2014. All rights reserved.</small></p>
</body>
</html>
