<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_07) on Mon Oct 08 15:56:47 CEST 2012 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>GenomeList (jbiowh 2.0.6-SNAPSHOT API)</title>
<meta name="date" content="2012-10-08">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="GenomeList (jbiowh 2.0.6-SNAPSHOT API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/GenomeList.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev Class</li>
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomePTTComparator.html" title="class in org.jbiowh.core.datasets.gene.utils"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowh/core/datasets/gene/utils/GenomeList.html" target="_top">Frames</a></li>
<li><a href="GenomeList.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<!-- ======== START OF CLASS DATA ======== -->
<div class="header">
<div class="subTitle">org.jbiowh.core.datasets.gene.utils</div>
<h2 title="Class GenomeList" class="title">Class GenomeList</h2>
</div>
<div class="contentContainer">
<ul class="inheritance">
<li><a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">java.lang.Object</a></li>
<li>
<ul class="inheritance">
<li>org.jbiowh.core.datasets.gene.utils.GenomeList</li>
</ul>
</li>
</ul>
<div class="description">
<ul class="blockList">
<li class="blockList">
<hr>
<br>
<pre>public class <span class="strong">GenomeList</span>
extends <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a></pre>
<div class="block">This Class handled Genomes PTT

 $Author: r78v10a07@gmail.com $ 
 $LastChangedDate: 2012-10-03 22:11:05 +0200 (Wed, 03 Oct 2012) $ 
 $LastChangedRevision: 270 $</div>
<dl><dt><span class="strong">Since:</span></dt>
  <dd>Aug 22, 2012</dd></dl>
</li>
</ul>
</div>
<div class="summary">
<ul class="blockList">
<li class="blockList">
<!-- ======== CONSTRUCTOR SUMMARY ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_summary">
<!--   -->
</a>
<h3>Constructor Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Constructor Summary table, listing constructors, and an explanation">
<caption><span>Constructors</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tr class="altColor">
<td class="colOne"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#GenomeList()">GenomeList</a></strong>()</code>
<div class="block">Creates the Genomes List (Chromosome)</div>
</td>
</tr>
</table>
</li>
</ul>
<!-- ========== METHOD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="method_summary">
<!--   -->
</a>
<h3>Method Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Method Summary table, listing methods, and an explanation">
<caption><span>Methods</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#createGenomesListFromGeneId(java.lang.String)">createGenomesListFromGeneId</a></strong>(<a href="http://docs.oracle.com/javase/7/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;geneId)</code>
<div class="block">Create a Genome List (chromosome) from one of its genes</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#createGenomesListFromGeneInfo(org.jbiowh.core.datasets.gene.jpa.entities.GeneInfo)">createGenomesListFromGeneInfo</a></strong>(<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GeneInfo.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GeneInfo</a>&nbsp;gene)</code>
<div class="block">Create a Genome List (chromosome) from one of its genes</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#createGenomesListFromGeneInfo(java.lang.Long)">createGenomesListFromGeneInfo</a></strong>(<a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Long.html?is-external=true" title="class or interface in java.lang">Long</a>&nbsp;geneId)</code>
<div class="block">Create a Genome List (chromosome) from one of its genes</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#createGenomesListFromGenomePTT(org.jbiowh.core.datasets.gene.jpa.entities.GenomesPTT)">createGenomesListFromGenomePTT</a></strong>(<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&nbsp;genome)</code>
<div class="block">Create a Genome List (chromosome) from one of its genes</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#createGenomesListFromProteinGi(java.lang.String)">createGenomesListFromProteinGi</a></strong>(<a href="http://docs.oracle.com/javase/7/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;proteinGi)</code>
<div class="block">Create a Genome List (chromosome) from one of its proteinGi</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#createGenomesListFromPTTFile(java.lang.String)">createGenomesListFromPTTFile</a></strong>(<a href="http://docs.oracle.com/javase/7/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;pttFile)</code>
<div class="block">Create a Genome List (chromosome) from one its PTT file name</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a></code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#getCurrent()">getCurrent</a></strong>()</code>
<div class="block">Get the current gene on the chromosome list</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a></code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#getFirst()">getFirst</a></strong>()</code>
<div class="block">Return the first gene on the chromosome</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#getGenePosition()">getGenePosition</a></strong>()</code>
<div class="block">Return the gene position in the Chromosome</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/7/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#getGenomes()">getGenomes</a></strong>()</code>
<div class="block">Return the list of genes on the chromosome</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a></code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#getLast()">getLast</a></strong>()</code>
<div class="block">Return the last gene on the chromosome</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a></code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#getNext()">getNext</a></strong>()</code>
<div class="block">Return the next gene to the current</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a></code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#getPrevious()">getPrevious</a></strong>()</code>
<div class="block">Return the previous gene to the current</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#isEmpty()">isEmpty</a></strong>()</code>
<div class="block">Returns true if this list contains no elements.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#orderByPFrom()">orderByPFrom</a></strong>()</code>
<div class="block">Order the genes by its position on the chromosome</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#setCurrent(org.jbiowh.core.datasets.gene.jpa.entities.GenomesPTT)">setCurrent</a></strong>(<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&nbsp;genome)</code>
<div class="block">Set the current gene to the GenomesPTT (gene) object</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#setCurrent(int)">setCurrent</a></strong>(int&nbsp;index)</code>
<div class="block">Set the index position to the index</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#setGenomes(java.util.List)">setGenomes</a></strong>(<a href="http://docs.oracle.com/javase/7/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&gt;&nbsp;genomes)</code>
<div class="block">Set the list of genes on the chromosome.</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomeList.html#size()">size</a></strong>()</code>
<div class="block">Return the amount of genes on the chromosome</div>
</td>
</tr>
</table>
<ul class="blockList">
<li class="blockList"><a name="methods_inherited_from_class_java.lang.Object">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;java.lang.<a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a></h3>
<code><a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#clone()" title="class or interface in java.lang">clone</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#equals(java.lang.Object)" title="class or interface in java.lang">equals</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#finalize()" title="class or interface in java.lang">finalize</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#getClass()" title="class or interface in java.lang">getClass</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#hashCode()" title="class or interface in java.lang">hashCode</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#notify()" title="class or interface in java.lang">notify</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#notifyAll()" title="class or interface in java.lang">notifyAll</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#toString()" title="class or interface in java.lang">toString</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#wait()" title="class or interface in java.lang">wait</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#wait(long)" title="class or interface in java.lang">wait</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#wait(long, int)" title="class or interface in java.lang">wait</a></code></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="details">
<ul class="blockList">
<li class="blockList">
<!-- ========= CONSTRUCTOR DETAIL ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_detail">
<!--   -->
</a>
<h3>Constructor Detail</h3>
<a name="GenomeList()">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>GenomeList</h4>
<pre>public&nbsp;GenomeList()</pre>
<div class="block">Creates the Genomes List (Chromosome)</div>
</li>
</ul>
</li>
</ul>
<!-- ============ METHOD DETAIL ========== -->
<ul class="blockList">
<li class="blockList"><a name="method_detail">
<!--   -->
</a>
<h3>Method Detail</h3>
<a name="size()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>size</h4>
<pre>public&nbsp;int&nbsp;size()</pre>
<div class="block">Return the amount of genes on the chromosome</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the amount of genes on the chromosome</dd></dl>
</li>
</ul>
<a name="createGenomesListFromGeneInfo(java.lang.Long)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>createGenomesListFromGeneInfo</h4>
<pre>public&nbsp;void&nbsp;createGenomesListFromGeneInfo(<a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Long.html?is-external=true" title="class or interface in java.lang">Long</a>&nbsp;geneId)
                                   throws <a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></pre>
<div class="block">Create a Genome List (chromosome) from one of its genes</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>geneId</code> - the gene Id to search the chromosome</dd>
<dt><span class="strong">Throws:</span></dt>
<dd><code><a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></code></dd></dl>
</li>
</ul>
<a name="createGenomesListFromGeneId(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>createGenomesListFromGeneId</h4>
<pre>public&nbsp;void&nbsp;createGenomesListFromGeneId(<a href="http://docs.oracle.com/javase/7/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;geneId)
                                 throws <a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></pre>
<div class="block">Create a Genome List (chromosome) from one of its genes</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>geneId</code> - the gene Id to search the chromosome</dd>
<dt><span class="strong">Throws:</span></dt>
<dd><code><a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></code></dd></dl>
</li>
</ul>
<a name="createGenomesListFromProteinGi(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>createGenomesListFromProteinGi</h4>
<pre>public&nbsp;void&nbsp;createGenomesListFromProteinGi(<a href="http://docs.oracle.com/javase/7/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;proteinGi)
                                    throws <a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></pre>
<div class="block">Create a Genome List (chromosome) from one of its proteinGi</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>proteinGi</code> - the gene Id to search the chromosome</dd>
<dt><span class="strong">Throws:</span></dt>
<dd><code><a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></code></dd></dl>
</li>
</ul>
<a name="getGenePosition()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGenePosition</h4>
<pre>public&nbsp;int&nbsp;getGenePosition()</pre>
<div class="block">Return the gene position in the Chromosome</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the gene position in the Chromosome</dd></dl>
</li>
</ul>
<a name="createGenomesListFromGeneInfo(org.jbiowh.core.datasets.gene.jpa.entities.GeneInfo)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>createGenomesListFromGeneInfo</h4>
<pre>public&nbsp;void&nbsp;createGenomesListFromGeneInfo(<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GeneInfo.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GeneInfo</a>&nbsp;gene)
                                   throws <a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></pre>
<div class="block">Create a Genome List (chromosome) from one of its genes</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>gene</code> - the gene object</dd>
<dt><span class="strong">Throws:</span></dt>
<dd><code><a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></code></dd></dl>
</li>
</ul>
<a name="createGenomesListFromGenomePTT(org.jbiowh.core.datasets.gene.jpa.entities.GenomesPTT)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>createGenomesListFromGenomePTT</h4>
<pre>public&nbsp;void&nbsp;createGenomesListFromGenomePTT(<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&nbsp;genome)
                                    throws <a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></pre>
<div class="block">Create a Genome List (chromosome) from one of its genes</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>genome</code> - the genome object</dd>
<dt><span class="strong">Throws:</span></dt>
<dd><code><a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></code></dd></dl>
</li>
</ul>
<a name="createGenomesListFromPTTFile(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>createGenomesListFromPTTFile</h4>
<pre>public&nbsp;void&nbsp;createGenomesListFromPTTFile(<a href="http://docs.oracle.com/javase/7/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;pttFile)
                                  throws <a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></pre>
<div class="block">Create a Genome List (chromosome) from one its PTT file name</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>pttFile</code> - the PTT file name</dd>
<dt><span class="strong">Throws:</span></dt>
<dd><code><a href="../../../../../../org/jbiowh/core/dbms/sql/exception/SQLConstrainException.html" title="class in org.jbiowh.core.dbms.sql.exception">SQLConstrainException</a></code></dd></dl>
</li>
</ul>
<a name="getGenomes()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGenomes</h4>
<pre>public&nbsp;<a href="http://docs.oracle.com/javase/7/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&gt;&nbsp;getGenomes()</pre>
<div class="block">Return the list of genes on the chromosome</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the list of genes on the chromosome</dd></dl>
</li>
</ul>
<a name="isEmpty()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>isEmpty</h4>
<pre>public&nbsp;boolean&nbsp;isEmpty()</pre>
<div class="block">Returns true if this list contains no elements.</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>true if this list contains no elements.</dd></dl>
</li>
</ul>
<a name="setGenomes(java.util.List)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGenomes</h4>
<pre>public&nbsp;void&nbsp;setGenomes(<a href="http://docs.oracle.com/javase/7/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&gt;&nbsp;genomes)</pre>
<div class="block">Set the list of genes on the chromosome. Set the current gene to the
 first gene on the chromosome</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>genomes</code> - the list of genes on the chromosome.</dd></dl>
</li>
</ul>
<a name="orderByPFrom()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>orderByPFrom</h4>
<pre>public&nbsp;void&nbsp;orderByPFrom()</pre>
<div class="block">Order the genes by its position on the chromosome</div>
</li>
</ul>
<a name="getFirst()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getFirst</h4>
<pre>public&nbsp;<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&nbsp;getFirst()</pre>
<div class="block">Return the first gene on the chromosome</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the first gene on the chromosome</dd></dl>
</li>
</ul>
<a name="getLast()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getLast</h4>
<pre>public&nbsp;<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&nbsp;getLast()</pre>
<div class="block">Return the last gene on the chromosome</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the last gene on the chromosome</dd></dl>
</li>
</ul>
<a name="getCurrent()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getCurrent</h4>
<pre>public&nbsp;<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&nbsp;getCurrent()</pre>
<div class="block">Get the current gene on the chromosome list</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the current gene on the chromosome list</dd></dl>
</li>
</ul>
<a name="setCurrent(int)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setCurrent</h4>
<pre>public&nbsp;int&nbsp;setCurrent(int&nbsp;index)</pre>
<div class="block">Set the index position to the index</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>index</code> - the index position</dd>
<dt><span class="strong">Returns:</span></dt><dd>-1 in an error occurs, the index if its an OK operation</dd></dl>
</li>
</ul>
<a name="setCurrent(org.jbiowh.core.datasets.gene.jpa.entities.GenomesPTT)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setCurrent</h4>
<pre>public&nbsp;int&nbsp;setCurrent(<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&nbsp;genome)</pre>
<div class="block">Set the current gene to the GenomesPTT (gene) object</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>genome</code> - the GenomesPTT (gene) object</dd>
<dt><span class="strong">Returns:</span></dt><dd>-1 in an error occurs, the index if its an OK operation</dd></dl>
</li>
</ul>
<a name="getNext()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getNext</h4>
<pre>public&nbsp;<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&nbsp;getNext()</pre>
<div class="block">Return the next gene to the current</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the next gene to the current</dd></dl>
</li>
</ul>
<a name="getPrevious()">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>getPrevious</h4>
<pre>public&nbsp;<a href="../../../../../../org/jbiowh/core/datasets/gene/jpa/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.jpa.entities">GenomesPTT</a>&nbsp;getPrevious()</pre>
<div class="block">Return the previous gene to the current</div>
<dl><dt><span class="strong">Returns:</span></dt><dd>the previous gene to the current</dd></dl>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- ========= END OF CLASS DATA ========= -->
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/GenomeList.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev Class</li>
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/utils/GenomePTTComparator.html" title="class in org.jbiowh.core.datasets.gene.utils"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowh/core/datasets/gene/utils/GenomeList.html" target="_top">Frames</a></li>
<li><a href="GenomeList.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2012. All Rights Reserved.</small></p>
</body>
</html>
