<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_21) on Wed May 29 12:16:42 CEST 2013 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>ProteinProteinGraph (jbiowh 3.1.0-SNAPSHOT API)</title>
<meta name="date" content="2013-05-29">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="ProteinProteinGraph (jbiowh 3.1.0-SNAPSHOT API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/ProteinProteinGraph.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../org/jbiowh/core/datasets/protein/utils/ProteinFromOther.html" title="class in org.jbiowh.core.datasets.protein.utils"><span class="strong">Prev Class</span></a></li>
<li><a href="../../../../../../org/jbiowh/core/datasets/protein/utils/ProteinToFasta.html" title="class in org.jbiowh.core.datasets.protein.utils"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowh/core/datasets/protein/utils/ProteinProteinGraph.html" target="_top">Frames</a></li>
<li><a href="ProteinProteinGraph.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#fields_inherited_from_class_org.jbiowh.core.utility.graph.JBioWHGraph">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<!-- ======== START OF CLASS DATA ======== -->
<div class="header">
<div class="subTitle">org.jbiowh.core.datasets.protein.utils</div>
<h2 title="Class ProteinProteinGraph" class="title">Class ProteinProteinGraph</h2>
</div>
<div class="contentContainer">
<ul class="inheritance">
<li><a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">java.lang.Object</a></li>
<li>
<ul class="inheritance">
<li><a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html" title="class in org.jbiowh.core.utility.graph">org.jbiowh.core.utility.graph.JBioWHGraph</a></li>
<li>
<ul class="inheritance">
<li>org.jbiowh.core.datasets.protein.utils.ProteinProteinGraph</li>
</ul>
</li>
</ul>
</li>
</ul>
<div class="description">
<ul class="blockList">
<li class="blockList">
<hr>
<br>
<pre>public class <span class="strong">ProteinProteinGraph</span>
extends <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html" title="class in org.jbiowh.core.utility.graph">JBioWHGraph</a></pre>
<div class="block">This class is creates a Protein-protein interaction graph

 $Author$ $LastChangedDate$ $LastChangedRevision$</div>
<dl><dt><span class="strong">Since:</span></dt>
  <dd>Mar 28, 2013</dd></dl>
</li>
</ul>
</div>
<div class="summary">
<ul class="blockList">
<li class="blockList">
<!-- =========== FIELD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="field_summary">
<!--   -->
</a>
<h3>Field Summary</h3>
<ul class="blockList">
<li class="blockList"><a name="fields_inherited_from_class_org.jbiowh.core.utility.graph.JBioWHGraph">
<!--   -->
</a>
<h3>Fields inherited from class&nbsp;org.jbiowh.core.utility.graph.<a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html" title="class in org.jbiowh.core.utility.graph">JBioWHGraph</a></h3>
<code><a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#depth">depth</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#graph">graph</a></code></li>
</ul>
</li>
</ul>
<!-- ======== CONSTRUCTOR SUMMARY ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_summary">
<!--   -->
</a>
<h3>Constructor Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Constructor Summary table, listing constructors, and an explanation">
<caption><span>Constructors</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tr class="altColor">
<td class="colOne"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/protein/utils/ProteinProteinGraph.html#ProteinProteinGraph(int)">ProteinProteinGraph</a></strong>(int&nbsp;depth)</code>
<div class="block">Creates a ProteinProteinGraph object where the graph object is a weighted
 graph.</div>
</td>
</tr>
</table>
</li>
</ul>
<!-- ========== METHOD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="method_summary">
<!--   -->
</a>
<h3>Method Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Method Summary table, listing methods, and an explanation">
<caption><span>Methods</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/protein/utils/ProteinProteinGraph.html#populate(org.jbiowh.core.datasets.protein.jpa.entities.Protein, int)">populate</a></strong>(<a href="../../../../../../org/jbiowh/core/datasets/protein/jpa/entities/Protein.html" title="class in org.jbiowh.core.datasets.protein.jpa.entities">Protein</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from Protein entity.</div>
</td>
</tr>
</table>
<ul class="blockList">
<li class="blockList"><a name="methods_inherited_from_class_org.jbiowh.core.utility.graph.JBioWHGraph">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;org.jbiowh.core.utility.graph.<a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html" title="class in org.jbiowh.core.utility.graph">JBioWHGraph</a></h3>
<code><a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#addBioWHNewVertex(java.lang.Object, java.lang.Object, int)">addBioWHNewVertex</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#addBioWHNewVertex(java.lang.Object, java.lang.Object, int, java.lang.Object)">addBioWHNewVertex</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#containsVertex(java.lang.Object)">containsVertex</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#edgeCount()">edgeCount</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#edgeToString(java.lang.Object, java.lang.Object, int)">edgeToString</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#getGraph()">getGraph</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#getGraphDijkstraShortestPath(java.lang.Object, java.lang.Object)">getGraphDijkstraShortestPath</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#getPathInList()">getPathInList</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#getPathInList(java.util.Collection)">getPathInList</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#isGraphConnected()">isGraphConnected</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.drug.drugbank.jpa.entities.DrugBank, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.gene.genebank.jpa.entities.GeneBankCDS, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.gene.genebank.jpa.entities.GeneBank, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.gene.gene.jpa.entities.GeneInfo, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.gene.genome.jpa.entities.GenePTT, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.pathway.kegg.jpa.entities.compound.KEGGCompound, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.pathway.kegg.jpa.entities.enzyme.KEGGEnzyme, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.pathway.kegg.jpa.entities.gene.KEGGGene, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.pathway.kegg.jpa.entities.glycan.KEGGGlycan, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.pathway.kegg.jpa.entities.pathway.KEGGPathway, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.pathway.kegg.jpa.entities.reaction.KEGGReaction, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.ppi.jpa.entities.MIFEntryInteraction, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.ppi.jpa.entities.MIFEntrySetEntry, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.ppi.jpa.entities.MIFEntrySet, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(java.lang.Object)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(java.lang.Object, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.disease.omim.jpa.entities.OMIM, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.domain.pfam.jpa.entities.PfamAbioWH, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.protclust.jpa.entities.UniRefEntry, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.protclust.jpa.entities.UniRefMember, int)">populate</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populateCollection(java.util.Collection, java.lang.Object, int)">populateCollection</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populateCollection(java.util.Collection, java.lang.Object, int, java.lang.Object)">populateCollection</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#tabToString(int)">tabToString</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#vertexCount()">vertexCount</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#vertexSet()">vertexSet</a>, <a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#vertexToString(java.lang.Object, int)">vertexToString</a></code></li>
</ul>
<ul class="blockList">
<li class="blockList"><a name="methods_inherited_from_class_java.lang.Object">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;java.lang.<a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true" title="class or interface in java.lang">Object</a></h3>
<code><a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#clone()" title="class or interface in java.lang">clone</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#equals(java.lang.Object)" title="class or interface in java.lang">equals</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#finalize()" title="class or interface in java.lang">finalize</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#getClass()" title="class or interface in java.lang">getClass</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#hashCode()" title="class or interface in java.lang">hashCode</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#notify()" title="class or interface in java.lang">notify</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#notifyAll()" title="class or interface in java.lang">notifyAll</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#toString()" title="class or interface in java.lang">toString</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#wait()" title="class or interface in java.lang">wait</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#wait(long)" title="class or interface in java.lang">wait</a>, <a href="http://docs.oracle.com/javase/7/docs/api/java/lang/Object.html?is-external=true#wait(long, int)" title="class or interface in java.lang">wait</a></code></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="details">
<ul class="blockList">
<li class="blockList">
<!-- ========= CONSTRUCTOR DETAIL ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_detail">
<!--   -->
</a>
<h3>Constructor Detail</h3>
<a name="ProteinProteinGraph(int)">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>ProteinProteinGraph</h4>
<pre>public&nbsp;ProteinProteinGraph(int&nbsp;depth)</pre>
<div class="block">Creates a ProteinProteinGraph object where the graph object is a weighted
 graph. The weights represents how many times is present the interaction
 into the MIF25 data sources.

 The graph uses the class HideWeightedEdge as edge.</div>
<dl><dt><span class="strong">Parameters:</span></dt><dd><code>depth</code> - the recursivity depth limit</dd><dt><span class="strong">See Also:</span></dt><dd><a href="../../../../../../org/jbiowh/core/utility/graph/edge/HideWeightedEdge.html" title="class in org.jbiowh.core.utility.graph.edge"><code>HideWeightedEdge</code></a></dd></dl>
</li>
</ul>
</li>
</ul>
<!-- ============ METHOD DETAIL ========== -->
<ul class="blockList">
<li class="blockList"><a name="method_detail">
<!--   -->
</a>
<h3>Method Detail</h3>
<a name="populate(org.jbiowh.core.datasets.protein.jpa.entities.Protein, int)">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>populate</h4>
<pre>protected&nbsp;void&nbsp;populate(<a href="../../../../../../org/jbiowh/core/datasets/protein/jpa/entities/Protein.html" title="class in org.jbiowh.core.datasets.protein.jpa.entities">Protein</a>&nbsp;o,
            int&nbsp;varDepth)</pre>
<div class="block">The recursive function to populate the graph from Protein entity. The
 MIFEntryInteraction entity is used as edge.

 The method creates a weighted graph of protein-protein interaction where
 the vertex are proteins and the edges are the interaction published by
 the MIF25 data sources.

 The weights represents how how many times is present the interaction into
 the MIF25 data sources.</div>
<dl>
<dt><strong>Overrides:</strong></dt>
<dd><code><a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html#populate(org.jbiowh.core.datasets.protein.jpa.entities.Protein, int)">populate</a></code>&nbsp;in class&nbsp;<code><a href="../../../../../../org/jbiowh/core/utility/graph/JBioWHGraph.html" title="class in org.jbiowh.core.utility.graph">JBioWHGraph</a></code></dd>
<dt><span class="strong">Parameters:</span></dt><dd><code>o</code> - the Protein entity to start the graph</dd><dd><code>varDepth</code> - the recursive depth for parent level</dd></dl>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- ========= END OF CLASS DATA ========= -->
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/ProteinProteinGraph.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../org/jbiowh/core/datasets/protein/utils/ProteinFromOther.html" title="class in org.jbiowh.core.datasets.protein.utils"><span class="strong">Prev Class</span></a></li>
<li><a href="../../../../../../org/jbiowh/core/datasets/protein/utils/ProteinToFasta.html" title="class in org.jbiowh.core.datasets.protein.utils"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowh/core/datasets/protein/utils/ProteinProteinGraph.html" target="_top">Frames</a></li>
<li><a href="ProteinProteinGraph.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#fields_inherited_from_class_org.jbiowh.core.utility.graph.JBioWHGraph">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2013. All Rights Reserved.</small></p>
</body>
</html>
