<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_05) on Tue Jul 03 09:59:06 CEST 2012 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Gene2Accession_</title>
<meta name="date" content="2012-07-03">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Gene2Accession_";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/Gene2Accession_.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities"><span class="strong">Prev Class</span></a></li>
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Ensembl.html" title="class in org.jbiowh.core.datasets.gene.entities"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html" target="_top">Frames</a></li>
<li><a href="Gene2Accession_.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#field_summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#methods_inherited_from_class_java.lang.Object">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field_detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li>Method</li>
</ul>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<!-- ======== START OF CLASS DATA ======== -->
<div class="header">
<div class="subTitle">org.jbiowh.core.datasets.gene.entities</div>
<h2 title="Class Gene2Accession_" class="title">Class Gene2Accession_</h2>
</div>
<div class="contentContainer">
<ul class="inheritance">
<li>java.lang.Object</li>
<li>
<ul class="inheritance">
<li>org.jbiowh.core.datasets.gene.entities.Gene2Accession_</li>
</ul>
</li>
</ul>
<div class="description">
<ul class="blockList">
<li class="blockList">
<hr>
<br>
<pre>@Generated(value="EclipseLink-2.3.0.v20110604-r9504",
           date="2012-07-03T09:57:26")
public class <span class="strong">Gene2Accession_</span>
extends java.lang.Object</pre>
</li>
</ul>
</div>
<div class="summary">
<ul class="blockList">
<li class="blockList">
<!-- =========== FIELD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="field_summary">
<!--   -->
</a>
<h3>Field Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Field Summary table, listing fields, and an explanation">
<caption><span>Fields</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Field and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#assembly">assembly</a></strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#endPositionOnTheGenomicAccession">endPositionOnTheGenomicAccession</a></strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#geneInfo">geneInfo</a></strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.Long&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#geneInfoWID">geneInfoWID</a></strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.entities">GenomesPTT</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#genomesPTT">genomesPTT</a></strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#genomicNucleotideAccession">genomicNucleotideAccession</a></strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.Integer&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#genomicNucleotideAccessionVersion">genomicNucleotideAccessionVersion</a></strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.math.BigInteger&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#genomicNucleotideGi">genomicNucleotideGi</a></strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#orientation">orientation</a></strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#proteinAccession">proteinAccession</a></strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.Integer&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#proteinAccessionVersion">proteinAccessionVersion</a></strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.math.BigInteger&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#proteinGi">proteinGi</a></strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#rNANucleotideAccession">rNANucleotideAccession</a></strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.Integer&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#rNANucleotideAccessionVersion">rNANucleotideAccessionVersion</a></strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.math.BigInteger&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#rNANucleotideGi">rNANucleotideGi</a></strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#startPositionOnTheGenomicAccession">startPositionOnTheGenomicAccession</a></strong></code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#status">status</a></strong></code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>static javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.Long&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#wid">wid</a></strong></code>&nbsp;</td>
</tr>
</table>
</li>
</ul>
<!-- ======== CONSTRUCTOR SUMMARY ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_summary">
<!--   -->
</a>
<h3>Constructor Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Constructor Summary table, listing constructors, and an explanation">
<caption><span>Constructors</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tr class="altColor">
<td class="colOne"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html#Gene2Accession_()">Gene2Accession_</a></strong>()</code>&nbsp;</td>
</tr>
</table>
</li>
</ul>
<!-- ========== METHOD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="method_summary">
<!--   -->
</a>
<h3>Method Summary</h3>
<ul class="blockList">
<li class="blockList"><a name="methods_inherited_from_class_java.lang.Object">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;java.lang.Object</h3>
<code>clone, equals, finalize, getClass, hashCode, notify, notifyAll, toString, wait, wait, wait</code></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="details">
<ul class="blockList">
<li class="blockList">
<!-- ============ FIELD DETAIL =========== -->
<ul class="blockList">
<li class="blockList"><a name="field_detail">
<!--   -->
</a>
<h3>Field Detail</h3>
<a name="orientation">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>orientation</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt; orientation</pre>
</li>
</ul>
<a name="genomicNucleotideAccessionVersion">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>genomicNucleotideAccessionVersion</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.Integer&gt; genomicNucleotideAccessionVersion</pre>
</li>
</ul>
<a name="wid">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>wid</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.Long&gt; wid</pre>
</li>
</ul>
<a name="status">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>status</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt; status</pre>
</li>
</ul>
<a name="assembly">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>assembly</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt; assembly</pre>
</li>
</ul>
<a name="genomicNucleotideGi">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>genomicNucleotideGi</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.math.BigInteger&gt; genomicNucleotideGi</pre>
</li>
</ul>
<a name="endPositionOnTheGenomicAccession">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>endPositionOnTheGenomicAccession</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt; endPositionOnTheGenomicAccession</pre>
</li>
</ul>
<a name="rNANucleotideGi">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>rNANucleotideGi</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.math.BigInteger&gt; rNANucleotideGi</pre>
</li>
</ul>
<a name="geneInfoWID">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>geneInfoWID</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.Long&gt; geneInfoWID</pre>
</li>
</ul>
<a name="proteinGi">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>proteinGi</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.math.BigInteger&gt; proteinGi</pre>
</li>
</ul>
<a name="genomicNucleotideAccession">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>genomicNucleotideAccession</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt; genomicNucleotideAccession</pre>
</li>
</ul>
<a name="proteinAccessionVersion">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>proteinAccessionVersion</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.Integer&gt; proteinAccessionVersion</pre>
</li>
</ul>
<a name="proteinAccession">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>proteinAccession</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt; proteinAccession</pre>
</li>
</ul>
<a name="rNANucleotideAccessionVersion">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>rNANucleotideAccessionVersion</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.Integer&gt; rNANucleotideAccessionVersion</pre>
</li>
</ul>
<a name="geneInfo">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>geneInfo</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfo</a>&gt; geneInfo</pre>
</li>
</ul>
<a name="rNANucleotideAccession">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>rNANucleotideAccession</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt; rNANucleotideAccession</pre>
</li>
</ul>
<a name="startPositionOnTheGenomicAccession">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>startPositionOnTheGenomicAccession</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,java.lang.String&gt; startPositionOnTheGenomicAccession</pre>
</li>
</ul>
<a name="genomesPTT">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>genomesPTT</h4>
<pre>public static volatile&nbsp;javax.persistence.metamodel.SingularAttribute&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.entities">GenomesPTT</a>&gt; genomesPTT</pre>
</li>
</ul>
</li>
</ul>
<!-- ========= CONSTRUCTOR DETAIL ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_detail">
<!--   -->
</a>
<h3>Constructor Detail</h3>
<a name="Gene2Accession_()">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>Gene2Accession_</h4>
<pre>public&nbsp;Gene2Accession_()</pre>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- ========= END OF CLASS DATA ========= -->
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/Gene2Accession_.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities"><span class="strong">Prev Class</span></a></li>
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Ensembl.html" title="class in org.jbiowh.core.datasets.gene.entities"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowh/core/datasets/gene/entities/Gene2Accession_.html" target="_top">Frames</a></li>
<li><a href="Gene2Accession_.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li><a href="#field_summary">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#methods_inherited_from_class_java.lang.Object">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li><a href="#field_detail">Field</a>&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li>Method</li>
</ul>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</body>
</html>
