<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_05) on Tue Jul 03 14:51:58 CEST 2012 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Gene2Accession</title>
<meta name="date" content="2012-07-03">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Gene2Accession";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/Gene2Accession.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev Class</li>
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Ensembl.html" title="class in org.jbiowh.core.datasets.gene.entities"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" target="_top">Frames</a></li>
<li><a href="Gene2Accession.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<!-- ======== START OF CLASS DATA ======== -->
<div class="header">
<div class="subTitle">org.jbiowh.core.datasets.gene.entities</div>
<h2 title="Class Gene2Accession" class="title">Class Gene2Accession</h2>
</div>
<div class="contentContainer">
<ul class="inheritance">
<li>java.lang.Object</li>
<li>
<ul class="inheritance">
<li>org.jbiowh.core.datasets.gene.entities.Gene2Accession</li>
</ul>
</li>
</ul>
<div class="description">
<ul class="blockList">
<li class="blockList">
<dl>
<dt>All Implemented Interfaces:</dt>
<dd>java.io.Serializable</dd>
</dl>
<hr>
<br>
<pre>@Entity
public class <span class="strong">Gene2Accession</span>
extends java.lang.Object
implements java.io.Serializable</pre>
<div class="block">This Class is the Gene2Accession entity</div>
<dl><dt><span class="strong">Since:</span></dt>
  <dd>Jul 27, 2011</dd>
<dt><span class="strong">Version:</span></dt>
  <dd>1.0</dd>
<dt><span class="strong">Author:</span></dt>
  <dd>rvera</dd>
<dt><span class="strong">See Also:</span></dt><dd><a href="../../../../../../serialized-form.html#org.jbiowh.core.datasets.gene.entities.Gene2Accession">Serialized Form</a></dd></dl>
</li>
</ul>
</div>
<div class="summary">
<ul class="blockList">
<li class="blockList">
<!-- ======== CONSTRUCTOR SUMMARY ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_summary">
<!--   -->
</a>
<h3>Constructor Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Constructor Summary table, listing constructors, and an explanation">
<caption><span>Constructors</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tr class="altColor">
<td class="colOne"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#Gene2Accession()">Gene2Accession</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colOne"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#Gene2Accession(java.lang.Long)">Gene2Accession</a></strong>(java.lang.Long&nbsp;wid)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colOne"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#Gene2Accession(java.lang.Long, long)">Gene2Accession</a></strong>(java.lang.Long&nbsp;wid,
              long&nbsp;geneInfoWID)</code>&nbsp;</td>
</tr>
</table>
</li>
</ul>
<!-- ========== METHOD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="method_summary">
<!--   -->
</a>
<h3>Method Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Method Summary table, listing methods, and an explanation">
<caption><span>Methods</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#equals(java.lang.Object)">equals</a></strong>(java.lang.Object&nbsp;object)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getAssembly()">getAssembly</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getEndPositionOnTheGenomicAccession()">getEndPositionOnTheGenomicAccession</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfo</a></code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getGeneInfo()">getGeneInfo</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getGeneInfoWID()">getGeneInfoWID</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.entities">GenomesPTT</a></code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getGenomesPTT()">getGenomesPTT</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getGenomicNucleotideAccession()">getGenomicNucleotideAccession</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.Integer</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getGenomicNucleotideAccessionVersion()">getGenomicNucleotideAccessionVersion</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.math.BigInteger</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getGenomicNucleotideGi()">getGenomicNucleotideGi</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getOrientation()">getOrientation</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getProteinAccession()">getProteinAccession</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.Integer</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getProteinAccessionVersion()">getProteinAccessionVersion</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.math.BigInteger</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getProteinGi()">getProteinGi</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getrNANucleotideAccession()">getrNANucleotideAccession</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getRNANucleotideAccession()">getRNANucleotideAccession</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.Integer</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getrNANucleotideAccessionVersion()">getrNANucleotideAccessionVersion</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.Integer</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getRNANucleotideAccessionVersion()">getRNANucleotideAccessionVersion</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.math.BigInteger</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getrNANucleotideGi()">getrNANucleotideGi</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.math.BigInteger</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getRNANucleotideGi()">getRNANucleotideGi</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getStartPositionOnTheGenomicAccession()">getStartPositionOnTheGenomicAccession</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getStatus()">getStatus</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.Long</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#getWid()">getWid</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#hashCode()">hashCode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setAssembly(java.lang.String)">setAssembly</a></strong>(java.lang.String&nbsp;assembly)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setEndPositionOnTheGenomicAccession(java.lang.String)">setEndPositionOnTheGenomicAccession</a></strong>(java.lang.String&nbsp;endPositionOnTheGenomicAccession)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setGeneInfo(org.jbiowh.core.datasets.gene.entities.GeneInfo)">setGeneInfo</a></strong>(<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfo</a>&nbsp;geneInfo)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setGeneInfoWID(long)">setGeneInfoWID</a></strong>(long&nbsp;geneInfoWID)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setGenomesPTT(org.jbiowh.core.datasets.gene.entities.GenomesPTT)">setGenomesPTT</a></strong>(<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.entities">GenomesPTT</a>&nbsp;genomesPTT)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setGenomicNucleotideAccession(java.lang.String)">setGenomicNucleotideAccession</a></strong>(java.lang.String&nbsp;genomicNucleotideAccession)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setGenomicNucleotideAccessionVersion(java.lang.Integer)">setGenomicNucleotideAccessionVersion</a></strong>(java.lang.Integer&nbsp;genomicNucleotideAccessionVersion)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setGenomicNucleotideGi(java.math.BigInteger)">setGenomicNucleotideGi</a></strong>(java.math.BigInteger&nbsp;genomicNucleotideGi)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setOrientation(java.lang.String)">setOrientation</a></strong>(java.lang.String&nbsp;orientation)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setProteinAccession(java.lang.String)">setProteinAccession</a></strong>(java.lang.String&nbsp;proteinAccession)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setProteinAccessionVersion(java.lang.Integer)">setProteinAccessionVersion</a></strong>(java.lang.Integer&nbsp;proteinAccessionVersion)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setProteinGi(java.math.BigInteger)">setProteinGi</a></strong>(java.math.BigInteger&nbsp;proteinGi)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setrNANucleotideAccession(java.lang.String)">setrNANucleotideAccession</a></strong>(java.lang.String&nbsp;rNANucleotideAccession)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setRNANucleotideAccession(java.lang.String)">setRNANucleotideAccession</a></strong>(java.lang.String&nbsp;rNANucleotideAccession)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setrNANucleotideAccessionVersion(java.lang.Integer)">setrNANucleotideAccessionVersion</a></strong>(java.lang.Integer&nbsp;rNANucleotideAccessionVersion)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setRNANucleotideAccessionVersion(java.lang.Integer)">setRNANucleotideAccessionVersion</a></strong>(java.lang.Integer&nbsp;rNANucleotideAccessionVersion)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setrNANucleotideGi(java.math.BigInteger)">setrNANucleotideGi</a></strong>(java.math.BigInteger&nbsp;rNANucleotideGi)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setRNANucleotideGi(java.math.BigInteger)">setRNANucleotideGi</a></strong>(java.math.BigInteger&nbsp;rNANucleotideGi)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setStartPositionOnTheGenomicAccession(java.lang.String)">setStartPositionOnTheGenomicAccession</a></strong>(java.lang.String&nbsp;startPositionOnTheGenomicAccession)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setStatus(java.lang.String)">setStatus</a></strong>(java.lang.String&nbsp;status)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#setWid(java.lang.Long)">setWid</a></strong>(java.lang.Long&nbsp;wid)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html#toString()">toString</a></strong>()</code>&nbsp;</td>
</tr>
</table>
<ul class="blockList">
<li class="blockList"><a name="methods_inherited_from_class_java.lang.Object">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;java.lang.Object</h3>
<code>clone, finalize, getClass, notify, notifyAll, wait, wait, wait</code></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="details">
<ul class="blockList">
<li class="blockList">
<!-- ========= CONSTRUCTOR DETAIL ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_detail">
<!--   -->
</a>
<h3>Constructor Detail</h3>
<a name="Gene2Accession()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>Gene2Accession</h4>
<pre>public&nbsp;Gene2Accession()</pre>
</li>
</ul>
<a name="Gene2Accession(java.lang.Long)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>Gene2Accession</h4>
<pre>public&nbsp;Gene2Accession(java.lang.Long&nbsp;wid)</pre>
</li>
</ul>
<a name="Gene2Accession(java.lang.Long, long)">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>Gene2Accession</h4>
<pre>public&nbsp;Gene2Accession(java.lang.Long&nbsp;wid,
              long&nbsp;geneInfoWID)</pre>
</li>
</ul>
</li>
</ul>
<!-- ============ METHOD DETAIL ========== -->
<ul class="blockList">
<li class="blockList"><a name="method_detail">
<!--   -->
</a>
<h3>Method Detail</h3>
<a name="getGenomesPTT()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGenomesPTT</h4>
<pre>public&nbsp;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.entities">GenomesPTT</a>&nbsp;getGenomesPTT()</pre>
</li>
</ul>
<a name="setGenomesPTT(org.jbiowh.core.datasets.gene.entities.GenomesPTT)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGenomesPTT</h4>
<pre>public&nbsp;void&nbsp;setGenomesPTT(<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GenomesPTT.html" title="class in org.jbiowh.core.datasets.gene.entities">GenomesPTT</a>&nbsp;genomesPTT)</pre>
</li>
</ul>
<a name="getGeneInfo()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGeneInfo</h4>
<pre>public&nbsp;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfo</a>&nbsp;getGeneInfo()</pre>
</li>
</ul>
<a name="setGeneInfo(org.jbiowh.core.datasets.gene.entities.GeneInfo)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGeneInfo</h4>
<pre>public&nbsp;void&nbsp;setGeneInfo(<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfo</a>&nbsp;geneInfo)</pre>
</li>
</ul>
<a name="getrNANucleotideAccession()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getrNANucleotideAccession</h4>
<pre>public&nbsp;java.lang.String&nbsp;getrNANucleotideAccession()</pre>
</li>
</ul>
<a name="setrNANucleotideAccession(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setrNANucleotideAccession</h4>
<pre>public&nbsp;void&nbsp;setrNANucleotideAccession(java.lang.String&nbsp;rNANucleotideAccession)</pre>
</li>
</ul>
<a name="getrNANucleotideAccessionVersion()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getrNANucleotideAccessionVersion</h4>
<pre>public&nbsp;java.lang.Integer&nbsp;getrNANucleotideAccessionVersion()</pre>
</li>
</ul>
<a name="setrNANucleotideAccessionVersion(java.lang.Integer)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setrNANucleotideAccessionVersion</h4>
<pre>public&nbsp;void&nbsp;setrNANucleotideAccessionVersion(java.lang.Integer&nbsp;rNANucleotideAccessionVersion)</pre>
</li>
</ul>
<a name="getrNANucleotideGi()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getrNANucleotideGi</h4>
<pre>public&nbsp;java.math.BigInteger&nbsp;getrNANucleotideGi()</pre>
</li>
</ul>
<a name="setrNANucleotideGi(java.math.BigInteger)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setrNANucleotideGi</h4>
<pre>public&nbsp;void&nbsp;setrNANucleotideGi(java.math.BigInteger&nbsp;rNANucleotideGi)</pre>
</li>
</ul>
<a name="getWid()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getWid</h4>
<pre>public&nbsp;java.lang.Long&nbsp;getWid()</pre>
</li>
</ul>
<a name="setWid(java.lang.Long)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setWid</h4>
<pre>public&nbsp;void&nbsp;setWid(java.lang.Long&nbsp;wid)</pre>
</li>
</ul>
<a name="getGeneInfoWID()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGeneInfoWID</h4>
<pre>public&nbsp;long&nbsp;getGeneInfoWID()</pre>
</li>
</ul>
<a name="setGeneInfoWID(long)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGeneInfoWID</h4>
<pre>public&nbsp;void&nbsp;setGeneInfoWID(long&nbsp;geneInfoWID)</pre>
</li>
</ul>
<a name="getStatus()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getStatus</h4>
<pre>public&nbsp;java.lang.String&nbsp;getStatus()</pre>
</li>
</ul>
<a name="setStatus(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setStatus</h4>
<pre>public&nbsp;void&nbsp;setStatus(java.lang.String&nbsp;status)</pre>
</li>
</ul>
<a name="getRNANucleotideAccession()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getRNANucleotideAccession</h4>
<pre>public&nbsp;java.lang.String&nbsp;getRNANucleotideAccession()</pre>
</li>
</ul>
<a name="setRNANucleotideAccession(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setRNANucleotideAccession</h4>
<pre>public&nbsp;void&nbsp;setRNANucleotideAccession(java.lang.String&nbsp;rNANucleotideAccession)</pre>
</li>
</ul>
<a name="getRNANucleotideAccessionVersion()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getRNANucleotideAccessionVersion</h4>
<pre>public&nbsp;java.lang.Integer&nbsp;getRNANucleotideAccessionVersion()</pre>
</li>
</ul>
<a name="setRNANucleotideAccessionVersion(java.lang.Integer)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setRNANucleotideAccessionVersion</h4>
<pre>public&nbsp;void&nbsp;setRNANucleotideAccessionVersion(java.lang.Integer&nbsp;rNANucleotideAccessionVersion)</pre>
</li>
</ul>
<a name="getRNANucleotideGi()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getRNANucleotideGi</h4>
<pre>public&nbsp;java.math.BigInteger&nbsp;getRNANucleotideGi()</pre>
</li>
</ul>
<a name="setRNANucleotideGi(java.math.BigInteger)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setRNANucleotideGi</h4>
<pre>public&nbsp;void&nbsp;setRNANucleotideGi(java.math.BigInteger&nbsp;rNANucleotideGi)</pre>
</li>
</ul>
<a name="getProteinAccession()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinAccession</h4>
<pre>public&nbsp;java.lang.String&nbsp;getProteinAccession()</pre>
</li>
</ul>
<a name="setProteinAccession(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setProteinAccession</h4>
<pre>public&nbsp;void&nbsp;setProteinAccession(java.lang.String&nbsp;proteinAccession)</pre>
</li>
</ul>
<a name="getProteinAccessionVersion()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinAccessionVersion</h4>
<pre>public&nbsp;java.lang.Integer&nbsp;getProteinAccessionVersion()</pre>
</li>
</ul>
<a name="setProteinAccessionVersion(java.lang.Integer)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setProteinAccessionVersion</h4>
<pre>public&nbsp;void&nbsp;setProteinAccessionVersion(java.lang.Integer&nbsp;proteinAccessionVersion)</pre>
</li>
</ul>
<a name="getProteinGi()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProteinGi</h4>
<pre>public&nbsp;java.math.BigInteger&nbsp;getProteinGi()</pre>
</li>
</ul>
<a name="setProteinGi(java.math.BigInteger)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setProteinGi</h4>
<pre>public&nbsp;void&nbsp;setProteinGi(java.math.BigInteger&nbsp;proteinGi)</pre>
</li>
</ul>
<a name="getGenomicNucleotideAccession()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGenomicNucleotideAccession</h4>
<pre>public&nbsp;java.lang.String&nbsp;getGenomicNucleotideAccession()</pre>
</li>
</ul>
<a name="setGenomicNucleotideAccession(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGenomicNucleotideAccession</h4>
<pre>public&nbsp;void&nbsp;setGenomicNucleotideAccession(java.lang.String&nbsp;genomicNucleotideAccession)</pre>
</li>
</ul>
<a name="getGenomicNucleotideAccessionVersion()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGenomicNucleotideAccessionVersion</h4>
<pre>public&nbsp;java.lang.Integer&nbsp;getGenomicNucleotideAccessionVersion()</pre>
</li>
</ul>
<a name="setGenomicNucleotideAccessionVersion(java.lang.Integer)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGenomicNucleotideAccessionVersion</h4>
<pre>public&nbsp;void&nbsp;setGenomicNucleotideAccessionVersion(java.lang.Integer&nbsp;genomicNucleotideAccessionVersion)</pre>
</li>
</ul>
<a name="getGenomicNucleotideGi()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGenomicNucleotideGi</h4>
<pre>public&nbsp;java.math.BigInteger&nbsp;getGenomicNucleotideGi()</pre>
</li>
</ul>
<a name="setGenomicNucleotideGi(java.math.BigInteger)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGenomicNucleotideGi</h4>
<pre>public&nbsp;void&nbsp;setGenomicNucleotideGi(java.math.BigInteger&nbsp;genomicNucleotideGi)</pre>
</li>
</ul>
<a name="getStartPositionOnTheGenomicAccession()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getStartPositionOnTheGenomicAccession</h4>
<pre>public&nbsp;java.lang.String&nbsp;getStartPositionOnTheGenomicAccession()</pre>
</li>
</ul>
<a name="setStartPositionOnTheGenomicAccession(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setStartPositionOnTheGenomicAccession</h4>
<pre>public&nbsp;void&nbsp;setStartPositionOnTheGenomicAccession(java.lang.String&nbsp;startPositionOnTheGenomicAccession)</pre>
</li>
</ul>
<a name="getEndPositionOnTheGenomicAccession()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getEndPositionOnTheGenomicAccession</h4>
<pre>public&nbsp;java.lang.String&nbsp;getEndPositionOnTheGenomicAccession()</pre>
</li>
</ul>
<a name="setEndPositionOnTheGenomicAccession(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setEndPositionOnTheGenomicAccession</h4>
<pre>public&nbsp;void&nbsp;setEndPositionOnTheGenomicAccession(java.lang.String&nbsp;endPositionOnTheGenomicAccession)</pre>
</li>
</ul>
<a name="getOrientation()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getOrientation</h4>
<pre>public&nbsp;java.lang.String&nbsp;getOrientation()</pre>
</li>
</ul>
<a name="setOrientation(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setOrientation</h4>
<pre>public&nbsp;void&nbsp;setOrientation(java.lang.String&nbsp;orientation)</pre>
</li>
</ul>
<a name="getAssembly()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getAssembly</h4>
<pre>public&nbsp;java.lang.String&nbsp;getAssembly()</pre>
</li>
</ul>
<a name="setAssembly(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setAssembly</h4>
<pre>public&nbsp;void&nbsp;setAssembly(java.lang.String&nbsp;assembly)</pre>
</li>
</ul>
<a name="hashCode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>hashCode</h4>
<pre>public&nbsp;int&nbsp;hashCode()</pre>
<dl>
<dt><strong>Overrides:</strong></dt>
<dd><code>hashCode</code>&nbsp;in class&nbsp;<code>java.lang.Object</code></dd>
</dl>
</li>
</ul>
<a name="equals(java.lang.Object)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>equals</h4>
<pre>public&nbsp;boolean&nbsp;equals(java.lang.Object&nbsp;object)</pre>
<dl>
<dt><strong>Overrides:</strong></dt>
<dd><code>equals</code>&nbsp;in class&nbsp;<code>java.lang.Object</code></dd>
</dl>
</li>
</ul>
<a name="toString()">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>toString</h4>
<pre>public&nbsp;java.lang.String&nbsp;toString()</pre>
<dl>
<dt><strong>Overrides:</strong></dt>
<dd><code>toString</code>&nbsp;in class&nbsp;<code>java.lang.Object</code></dd>
</dl>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- ========= END OF CLASS DATA ========= -->
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/Gene2Accession.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev Class</li>
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Ensembl.html" title="class in org.jbiowh.core.datasets.gene.entities"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" target="_top">Frames</a></li>
<li><a href="Gene2Accession.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</body>
</html>
