<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_05) on Tue Jul 03 11:42:29 CEST 2012 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>GeneInfo</title>
<meta name="date" content="2012-07-03">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="GeneInfo";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/GeneInfo.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneGroupPK.html" title="class in org.jbiowh.core.datasets.gene.entities"><span class="strong">Prev Class</span></a></li>
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoDBXrefs.html" title="class in org.jbiowh.core.datasets.gene.entities"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowh/core/datasets/gene/entities/GeneInfo.html" target="_top">Frames</a></li>
<li><a href="GeneInfo.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<!-- ======== START OF CLASS DATA ======== -->
<div class="header">
<div class="subTitle">org.jbiowh.core.datasets.gene.entities</div>
<h2 title="Class GeneInfo" class="title">Class GeneInfo</h2>
</div>
<div class="contentContainer">
<ul class="inheritance">
<li>java.lang.Object</li>
<li>
<ul class="inheritance">
<li>org.jbiowh.core.datasets.gene.entities.GeneInfo</li>
</ul>
</li>
</ul>
<div class="description">
<ul class="blockList">
<li class="blockList">
<dl>
<dt>All Implemented Interfaces:</dt>
<dd>java.io.Serializable</dd>
</dl>
<hr>
<br>
<pre>@Entity
public class <span class="strong">GeneInfo</span>
extends java.lang.Object
implements java.io.Serializable</pre>
<div class="block">This Class is the GeneInfo entity</div>
<dl><dt><span class="strong">Since:</span></dt>
  <dd>Jul 27, 2011</dd>
<dt><span class="strong">Version:</span></dt>
  <dd>1.0</dd>
<dt><span class="strong">Author:</span></dt>
  <dd>rvera</dd>
<dt><span class="strong">See Also:</span></dt><dd><a href="../../../../../../serialized-form.html#org.jbiowh.core.datasets.gene.entities.GeneInfo">Serialized Form</a></dd></dl>
</li>
</ul>
</div>
<div class="summary">
<ul class="blockList">
<li class="blockList">
<!-- ======== CONSTRUCTOR SUMMARY ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_summary">
<!--   -->
</a>
<h3>Constructor Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Constructor Summary table, listing constructors, and an explanation">
<caption><span>Constructors</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colOne" scope="col">Constructor and Description</th>
</tr>
<tr class="altColor">
<td class="colOne"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#GeneInfo()">GeneInfo</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colOne"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#GeneInfo(java.lang.Long)">GeneInfo</a></strong>(java.lang.Long&nbsp;wid)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colOne"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#GeneInfo(java.lang.Long, long, long, long)">GeneInfo</a></strong>(java.lang.Long&nbsp;wid,
        long&nbsp;geneID,
        long&nbsp;taxID,
        long&nbsp;dataSetWID)</code>&nbsp;</td>
</tr>
</table>
</li>
</ul>
<!-- ========== METHOD SUMMARY =========== -->
<ul class="blockList">
<li class="blockList"><a name="method_summary">
<!--   -->
</a>
<h3>Method Summary</h3>
<table class="overviewSummary" border="0" cellpadding="3" cellspacing="0" summary="Method Summary table, listing methods, and an explanation">
<caption><span>Methods</span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tr class="altColor">
<td class="colFirst"><code>boolean</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#equals(java.lang.Object)">equals</a></strong>(java.lang.Object&nbsp;object)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getChromosome()">getChromosome</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getDataSetWID()">getDataSetWID</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getDescription()">getDescription</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getFullNameFromNomenclatureAuthority()">getFullNameFromNomenclatureAuthority</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getGene2Accession()">getGene2Accession</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Ensembl.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Ensembl</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getGene2Ensembl()">getGene2Ensembl</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2MIMPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2MIMPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2MIM.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2MIM</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getGene2MIM()">getGene2MIM</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2PMIDPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2PMIDPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2PMID.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2PMID</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getGene2PMID()">getGene2PMID</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2STSPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2STSPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2STS.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2STS</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getGene2STS()">getGene2STS</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2UniGenePK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2UniGenePK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2UniGene.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2UniGene</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getGene2UniGene()">getGene2UniGene</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneGroupPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneGroupPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneGroup.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneGroup</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getGeneGroup()">getGeneGroup</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getGeneID()">getGeneID</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoDBXrefsPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoDBXrefsPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoDBXrefs.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoDBXrefs</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getGeneInfoDBXrefs()">getGeneInfoDBXrefs</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoSynonymsPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoSynonymsPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoSynonyms.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoSynonyms</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getGeneInfoSynonyms()">getGeneInfoSynonyms</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/pathway/kegg/entities/KEGGGene.html" title="class in org.jbiowh.core.datasets.pathway.kegg.entities">KEGGGene</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getkEGGGenes()">getkEGGGenes</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getLocusTag()">getLocusTag</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getMapLocation()">getMapLocation</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.Date</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getModificationDate()">getModificationDate</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getNomenclatureStatus()">getNomenclatureStatus</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/ontology/entities/Ontology.html" title="class in org.jbiowh.core.datasets.ontology.entities">Ontology</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getOntology()">getOntology</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getOtherDesignations()">getOtherDesignations</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/protein/entities/Protein.html" title="class in org.jbiowh.core.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getSymbol()">getSymbol</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getSymbolFromNomenclature()">getSymbolFromNomenclature</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>long</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getTaxID()">getTaxID</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowh/core/datasets/taxonomy/entities/Taxonomy.html" title="class in org.jbiowh.core.datasets.taxonomy.entities">Taxonomy</a></code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getTaxonomy()">getTaxonomy</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getTypeOfGene()">getTypeOfGene</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.Long</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#getWid()">getWid</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>int</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#hashCode()">hashCode</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setChromosome(java.lang.String)">setChromosome</a></strong>(java.lang.String&nbsp;chromosome)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setDataSetWID(long)">setDataSetWID</a></strong>(long&nbsp;dataSetWID)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setDescription(java.lang.String)">setDescription</a></strong>(java.lang.String&nbsp;description)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setFullNameFromNomenclatureAuthority(java.lang.String)">setFullNameFromNomenclatureAuthority</a></strong>(java.lang.String&nbsp;fullNameFromNomenclatureAuthority)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setGene2Accession(java.util.Set)">setGene2Accession</a></strong>(java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>&gt;&nbsp;gene2Accession)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setGene2Ensembl(java.util.Set)">setGene2Ensembl</a></strong>(java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Ensembl.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Ensembl</a>&gt;&nbsp;gene2Ensembl)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setGene2MIM(java.util.Map)">setGene2MIM</a></strong>(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2MIMPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2MIMPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2MIM.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2MIM</a>&gt;&nbsp;gene2MIM)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setGene2PMID(java.util.Map)">setGene2PMID</a></strong>(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2PMIDPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2PMIDPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2PMID.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2PMID</a>&gt;&nbsp;gene2PMID)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setGene2STS(java.util.Map)">setGene2STS</a></strong>(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2STSPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2STSPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2STS.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2STS</a>&gt;&nbsp;gene2STS)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setGene2UniGene(java.util.Map)">setGene2UniGene</a></strong>(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2UniGenePK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2UniGenePK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2UniGene.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2UniGene</a>&gt;&nbsp;gene2UniGene)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setGeneGroup(java.util.Map)">setGeneGroup</a></strong>(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneGroupPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneGroupPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneGroup.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneGroup</a>&gt;&nbsp;geneGroup)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setGeneID(long)">setGeneID</a></strong>(long&nbsp;geneID)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setGeneInfoDBXrefs(java.util.Map)">setGeneInfoDBXrefs</a></strong>(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoDBXrefsPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoDBXrefsPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoDBXrefs.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoDBXrefs</a>&gt;&nbsp;geneInfoDBXrefs)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setGeneInfoSynonyms(java.util.Map)">setGeneInfoSynonyms</a></strong>(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoSynonymsPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoSynonymsPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoSynonyms.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoSynonyms</a>&gt;&nbsp;geneInfoSynonyms)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setkEGGGenes(java.util.Set)">setkEGGGenes</a></strong>(java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/pathway/kegg/entities/KEGGGene.html" title="class in org.jbiowh.core.datasets.pathway.kegg.entities">KEGGGene</a>&gt;&nbsp;kEGGGenes)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setLocusTag(java.lang.String)">setLocusTag</a></strong>(java.lang.String&nbsp;locusTag)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setMapLocation(java.lang.String)">setMapLocation</a></strong>(java.lang.String&nbsp;mapLocation)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setModificationDate(java.util.Date)">setModificationDate</a></strong>(java.util.Date&nbsp;modificationDate)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setNomenclatureStatus(java.lang.String)">setNomenclatureStatus</a></strong>(java.lang.String&nbsp;nomenclatureStatus)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setOntology(java.util.Set)">setOntology</a></strong>(java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/ontology/entities/Ontology.html" title="class in org.jbiowh.core.datasets.ontology.entities">Ontology</a>&gt;&nbsp;ontology)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setOtherDesignations(java.lang.String)">setOtherDesignations</a></strong>(java.lang.String&nbsp;otherDesignations)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setProtein(java.util.Set)">setProtein</a></strong>(java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/protein/entities/Protein.html" title="class in org.jbiowh.core.datasets.protein.entities">Protein</a>&gt;&nbsp;protein)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setSymbol(java.lang.String)">setSymbol</a></strong>(java.lang.String&nbsp;symbol)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setSymbolFromNomenclature(java.lang.String)">setSymbolFromNomenclature</a></strong>(java.lang.String&nbsp;symbolFromNomenclature)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setTaxID(long)">setTaxID</a></strong>(long&nbsp;taxID)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setTaxonomy(org.jbiowh.core.datasets.taxonomy.entities.Taxonomy)">setTaxonomy</a></strong>(<a href="../../../../../../org/jbiowh/core/datasets/taxonomy/entities/Taxonomy.html" title="class in org.jbiowh.core.datasets.taxonomy.entities">Taxonomy</a>&nbsp;taxonomy)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setTypeOfGene(java.lang.String)">setTypeOfGene</a></strong>(java.lang.String&nbsp;typeOfGene)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#setWid(java.lang.Long)">setWid</a></strong>(java.lang.Long&nbsp;wid)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>java.lang.String</code></td>
<td class="colLast"><code><strong><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfo.html#toString()">toString</a></strong>()</code>&nbsp;</td>
</tr>
</table>
<ul class="blockList">
<li class="blockList"><a name="methods_inherited_from_class_java.lang.Object">
<!--   -->
</a>
<h3>Methods inherited from class&nbsp;java.lang.Object</h3>
<code>clone, finalize, getClass, notify, notifyAll, wait, wait, wait</code></li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
<div class="details">
<ul class="blockList">
<li class="blockList">
<!-- ========= CONSTRUCTOR DETAIL ======== -->
<ul class="blockList">
<li class="blockList"><a name="constructor_detail">
<!--   -->
</a>
<h3>Constructor Detail</h3>
<a name="GeneInfo()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>GeneInfo</h4>
<pre>public&nbsp;GeneInfo()</pre>
</li>
</ul>
<a name="GeneInfo(java.lang.Long)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>GeneInfo</h4>
<pre>public&nbsp;GeneInfo(java.lang.Long&nbsp;wid)</pre>
</li>
</ul>
<a name="GeneInfo(java.lang.Long, long, long, long)">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>GeneInfo</h4>
<pre>public&nbsp;GeneInfo(java.lang.Long&nbsp;wid,
        long&nbsp;geneID,
        long&nbsp;taxID,
        long&nbsp;dataSetWID)</pre>
</li>
</ul>
</li>
</ul>
<!-- ============ METHOD DETAIL ========== -->
<ul class="blockList">
<li class="blockList"><a name="method_detail">
<!--   -->
</a>
<h3>Method Detail</h3>
<a name="getkEGGGenes()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getkEGGGenes</h4>
<pre>public&nbsp;java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/pathway/kegg/entities/KEGGGene.html" title="class in org.jbiowh.core.datasets.pathway.kegg.entities">KEGGGene</a>&gt;&nbsp;getkEGGGenes()</pre>
</li>
</ul>
<a name="setkEGGGenes(java.util.Set)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setkEGGGenes</h4>
<pre>public&nbsp;void&nbsp;setkEGGGenes(java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/pathway/kegg/entities/KEGGGene.html" title="class in org.jbiowh.core.datasets.pathway.kegg.entities">KEGGGene</a>&gt;&nbsp;kEGGGenes)</pre>
</li>
</ul>
<a name="getProtein()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getProtein</h4>
<pre>public&nbsp;java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/protein/entities/Protein.html" title="class in org.jbiowh.core.datasets.protein.entities">Protein</a>&gt;&nbsp;getProtein()</pre>
</li>
</ul>
<a name="setProtein(java.util.Set)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setProtein</h4>
<pre>public&nbsp;void&nbsp;setProtein(java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/protein/entities/Protein.html" title="class in org.jbiowh.core.datasets.protein.entities">Protein</a>&gt;&nbsp;protein)</pre>
</li>
</ul>
<a name="getChromosome()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getChromosome</h4>
<pre>public&nbsp;java.lang.String&nbsp;getChromosome()</pre>
</li>
</ul>
<a name="setChromosome(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setChromosome</h4>
<pre>public&nbsp;void&nbsp;setChromosome(java.lang.String&nbsp;chromosome)</pre>
</li>
</ul>
<a name="getDataSetWID()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getDataSetWID</h4>
<pre>public&nbsp;long&nbsp;getDataSetWID()</pre>
</li>
</ul>
<a name="setDataSetWID(long)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setDataSetWID</h4>
<pre>public&nbsp;void&nbsp;setDataSetWID(long&nbsp;dataSetWID)</pre>
</li>
</ul>
<a name="getDescription()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getDescription</h4>
<pre>public&nbsp;java.lang.String&nbsp;getDescription()</pre>
</li>
</ul>
<a name="setDescription(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setDescription</h4>
<pre>public&nbsp;void&nbsp;setDescription(java.lang.String&nbsp;description)</pre>
</li>
</ul>
<a name="getFullNameFromNomenclatureAuthority()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getFullNameFromNomenclatureAuthority</h4>
<pre>public&nbsp;java.lang.String&nbsp;getFullNameFromNomenclatureAuthority()</pre>
</li>
</ul>
<a name="setFullNameFromNomenclatureAuthority(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setFullNameFromNomenclatureAuthority</h4>
<pre>public&nbsp;void&nbsp;setFullNameFromNomenclatureAuthority(java.lang.String&nbsp;fullNameFromNomenclatureAuthority)</pre>
</li>
</ul>
<a name="getGene2Accession()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGene2Accession</h4>
<pre>public&nbsp;java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>&gt;&nbsp;getGene2Accession()</pre>
</li>
</ul>
<a name="setGene2Accession(java.util.Set)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGene2Accession</h4>
<pre>public&nbsp;void&nbsp;setGene2Accession(java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Accession.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Accession</a>&gt;&nbsp;gene2Accession)</pre>
</li>
</ul>
<a name="getGene2Ensembl()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGene2Ensembl</h4>
<pre>public&nbsp;java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Ensembl.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Ensembl</a>&gt;&nbsp;getGene2Ensembl()</pre>
</li>
</ul>
<a name="setGene2Ensembl(java.util.Set)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGene2Ensembl</h4>
<pre>public&nbsp;void&nbsp;setGene2Ensembl(java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2Ensembl.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2Ensembl</a>&gt;&nbsp;gene2Ensembl)</pre>
</li>
</ul>
<a name="getGene2MIM()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGene2MIM</h4>
<pre>public&nbsp;java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2MIMPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2MIMPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2MIM.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2MIM</a>&gt;&nbsp;getGene2MIM()</pre>
</li>
</ul>
<a name="setGene2MIM(java.util.Map)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGene2MIM</h4>
<pre>public&nbsp;void&nbsp;setGene2MIM(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2MIMPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2MIMPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2MIM.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2MIM</a>&gt;&nbsp;gene2MIM)</pre>
</li>
</ul>
<a name="getGene2PMID()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGene2PMID</h4>
<pre>public&nbsp;java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2PMIDPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2PMIDPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2PMID.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2PMID</a>&gt;&nbsp;getGene2PMID()</pre>
</li>
</ul>
<a name="setGene2PMID(java.util.Map)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGene2PMID</h4>
<pre>public&nbsp;void&nbsp;setGene2PMID(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2PMIDPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2PMIDPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2PMID.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2PMID</a>&gt;&nbsp;gene2PMID)</pre>
</li>
</ul>
<a name="getGene2STS()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGene2STS</h4>
<pre>public&nbsp;java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2STSPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2STSPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2STS.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2STS</a>&gt;&nbsp;getGene2STS()</pre>
</li>
</ul>
<a name="setGene2STS(java.util.Map)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGene2STS</h4>
<pre>public&nbsp;void&nbsp;setGene2STS(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2STSPK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2STSPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2STS.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2STS</a>&gt;&nbsp;gene2STS)</pre>
</li>
</ul>
<a name="getGene2UniGene()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGene2UniGene</h4>
<pre>public&nbsp;java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2UniGenePK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2UniGenePK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2UniGene.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2UniGene</a>&gt;&nbsp;getGene2UniGene()</pre>
</li>
</ul>
<a name="setGene2UniGene(java.util.Map)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGene2UniGene</h4>
<pre>public&nbsp;void&nbsp;setGene2UniGene(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2UniGenePK.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2UniGenePK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/Gene2UniGene.html" title="class in org.jbiowh.core.datasets.gene.entities">Gene2UniGene</a>&gt;&nbsp;gene2UniGene)</pre>
</li>
</ul>
<a name="getGeneGroup()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGeneGroup</h4>
<pre>public&nbsp;java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneGroupPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneGroupPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneGroup.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneGroup</a>&gt;&nbsp;getGeneGroup()</pre>
</li>
</ul>
<a name="setGeneGroup(java.util.Map)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGeneGroup</h4>
<pre>public&nbsp;void&nbsp;setGeneGroup(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneGroupPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneGroupPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneGroup.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneGroup</a>&gt;&nbsp;geneGroup)</pre>
</li>
</ul>
<a name="getGeneID()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGeneID</h4>
<pre>public&nbsp;long&nbsp;getGeneID()</pre>
</li>
</ul>
<a name="setGeneID(long)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGeneID</h4>
<pre>public&nbsp;void&nbsp;setGeneID(long&nbsp;geneID)</pre>
</li>
</ul>
<a name="getGeneInfoDBXrefs()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGeneInfoDBXrefs</h4>
<pre>public&nbsp;java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoDBXrefsPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoDBXrefsPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoDBXrefs.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoDBXrefs</a>&gt;&nbsp;getGeneInfoDBXrefs()</pre>
</li>
</ul>
<a name="setGeneInfoDBXrefs(java.util.Map)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGeneInfoDBXrefs</h4>
<pre>public&nbsp;void&nbsp;setGeneInfoDBXrefs(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoDBXrefsPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoDBXrefsPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoDBXrefs.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoDBXrefs</a>&gt;&nbsp;geneInfoDBXrefs)</pre>
</li>
</ul>
<a name="getGeneInfoSynonyms()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getGeneInfoSynonyms</h4>
<pre>public&nbsp;java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoSynonymsPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoSynonymsPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoSynonyms.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoSynonyms</a>&gt;&nbsp;getGeneInfoSynonyms()</pre>
</li>
</ul>
<a name="setGeneInfoSynonyms(java.util.Map)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setGeneInfoSynonyms</h4>
<pre>public&nbsp;void&nbsp;setGeneInfoSynonyms(java.util.Map&lt;<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoSynonymsPK.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoSynonymsPK</a>,<a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoSynonyms.html" title="class in org.jbiowh.core.datasets.gene.entities">GeneInfoSynonyms</a>&gt;&nbsp;geneInfoSynonyms)</pre>
</li>
</ul>
<a name="getLocusTag()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getLocusTag</h4>
<pre>public&nbsp;java.lang.String&nbsp;getLocusTag()</pre>
</li>
</ul>
<a name="setLocusTag(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setLocusTag</h4>
<pre>public&nbsp;void&nbsp;setLocusTag(java.lang.String&nbsp;locusTag)</pre>
</li>
</ul>
<a name="getMapLocation()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getMapLocation</h4>
<pre>public&nbsp;java.lang.String&nbsp;getMapLocation()</pre>
</li>
</ul>
<a name="setMapLocation(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setMapLocation</h4>
<pre>public&nbsp;void&nbsp;setMapLocation(java.lang.String&nbsp;mapLocation)</pre>
</li>
</ul>
<a name="getModificationDate()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getModificationDate</h4>
<pre>public&nbsp;java.util.Date&nbsp;getModificationDate()</pre>
</li>
</ul>
<a name="setModificationDate(java.util.Date)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setModificationDate</h4>
<pre>public&nbsp;void&nbsp;setModificationDate(java.util.Date&nbsp;modificationDate)</pre>
</li>
</ul>
<a name="getNomenclatureStatus()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getNomenclatureStatus</h4>
<pre>public&nbsp;java.lang.String&nbsp;getNomenclatureStatus()</pre>
</li>
</ul>
<a name="setNomenclatureStatus(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setNomenclatureStatus</h4>
<pre>public&nbsp;void&nbsp;setNomenclatureStatus(java.lang.String&nbsp;nomenclatureStatus)</pre>
</li>
</ul>
<a name="getOntology()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getOntology</h4>
<pre>public&nbsp;java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/ontology/entities/Ontology.html" title="class in org.jbiowh.core.datasets.ontology.entities">Ontology</a>&gt;&nbsp;getOntology()</pre>
</li>
</ul>
<a name="setOntology(java.util.Set)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setOntology</h4>
<pre>public&nbsp;void&nbsp;setOntology(java.util.Set&lt;<a href="../../../../../../org/jbiowh/core/datasets/ontology/entities/Ontology.html" title="class in org.jbiowh.core.datasets.ontology.entities">Ontology</a>&gt;&nbsp;ontology)</pre>
</li>
</ul>
<a name="getOtherDesignations()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getOtherDesignations</h4>
<pre>public&nbsp;java.lang.String&nbsp;getOtherDesignations()</pre>
</li>
</ul>
<a name="setOtherDesignations(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setOtherDesignations</h4>
<pre>public&nbsp;void&nbsp;setOtherDesignations(java.lang.String&nbsp;otherDesignations)</pre>
</li>
</ul>
<a name="getSymbol()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getSymbol</h4>
<pre>public&nbsp;java.lang.String&nbsp;getSymbol()</pre>
</li>
</ul>
<a name="setSymbol(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setSymbol</h4>
<pre>public&nbsp;void&nbsp;setSymbol(java.lang.String&nbsp;symbol)</pre>
</li>
</ul>
<a name="getSymbolFromNomenclature()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getSymbolFromNomenclature</h4>
<pre>public&nbsp;java.lang.String&nbsp;getSymbolFromNomenclature()</pre>
</li>
</ul>
<a name="setSymbolFromNomenclature(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setSymbolFromNomenclature</h4>
<pre>public&nbsp;void&nbsp;setSymbolFromNomenclature(java.lang.String&nbsp;symbolFromNomenclature)</pre>
</li>
</ul>
<a name="getTaxID()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getTaxID</h4>
<pre>public&nbsp;long&nbsp;getTaxID()</pre>
</li>
</ul>
<a name="setTaxID(long)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setTaxID</h4>
<pre>public&nbsp;void&nbsp;setTaxID(long&nbsp;taxID)</pre>
</li>
</ul>
<a name="getTaxonomy()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getTaxonomy</h4>
<pre>public&nbsp;<a href="../../../../../../org/jbiowh/core/datasets/taxonomy/entities/Taxonomy.html" title="class in org.jbiowh.core.datasets.taxonomy.entities">Taxonomy</a>&nbsp;getTaxonomy()</pre>
</li>
</ul>
<a name="setTaxonomy(org.jbiowh.core.datasets.taxonomy.entities.Taxonomy)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setTaxonomy</h4>
<pre>public&nbsp;void&nbsp;setTaxonomy(<a href="../../../../../../org/jbiowh/core/datasets/taxonomy/entities/Taxonomy.html" title="class in org.jbiowh.core.datasets.taxonomy.entities">Taxonomy</a>&nbsp;taxonomy)</pre>
</li>
</ul>
<a name="getTypeOfGene()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getTypeOfGene</h4>
<pre>public&nbsp;java.lang.String&nbsp;getTypeOfGene()</pre>
</li>
</ul>
<a name="setTypeOfGene(java.lang.String)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setTypeOfGene</h4>
<pre>public&nbsp;void&nbsp;setTypeOfGene(java.lang.String&nbsp;typeOfGene)</pre>
</li>
</ul>
<a name="getWid()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>getWid</h4>
<pre>public&nbsp;java.lang.Long&nbsp;getWid()</pre>
</li>
</ul>
<a name="setWid(java.lang.Long)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>setWid</h4>
<pre>public&nbsp;void&nbsp;setWid(java.lang.Long&nbsp;wid)</pre>
</li>
</ul>
<a name="hashCode()">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>hashCode</h4>
<pre>public&nbsp;int&nbsp;hashCode()</pre>
<dl>
<dt><strong>Overrides:</strong></dt>
<dd><code>hashCode</code>&nbsp;in class&nbsp;<code>java.lang.Object</code></dd>
</dl>
</li>
</ul>
<a name="equals(java.lang.Object)">
<!--   -->
</a>
<ul class="blockList">
<li class="blockList">
<h4>equals</h4>
<pre>public&nbsp;boolean&nbsp;equals(java.lang.Object&nbsp;object)</pre>
<dl>
<dt><strong>Overrides:</strong></dt>
<dd><code>equals</code>&nbsp;in class&nbsp;<code>java.lang.Object</code></dd>
</dl>
</li>
</ul>
<a name="toString()">
<!--   -->
</a>
<ul class="blockListLast">
<li class="blockList">
<h4>toString</h4>
<pre>public&nbsp;java.lang.String&nbsp;toString()</pre>
<dl>
<dt><strong>Overrides:</strong></dt>
<dd><code>toString</code>&nbsp;in class&nbsp;<code>java.lang.Object</code></dd>
</dl>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</div>
</div>
<!-- ========= END OF CLASS DATA ========= -->
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="package-summary.html">Package</a></li>
<li class="navBarCell1Rev">Class</li>
<li><a href="class-use/GeneInfo.html">Use</a></li>
<li><a href="package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-files/index-1.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneGroupPK.html" title="class in org.jbiowh.core.datasets.gene.entities"><span class="strong">Prev Class</span></a></li>
<li><a href="../../../../../../org/jbiowh/core/datasets/gene/entities/GeneInfoDBXrefs.html" title="class in org.jbiowh.core.datasets.gene.entities"><span class="strong">Next Class</span></a></li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowh/core/datasets/gene/entities/GeneInfo.html" target="_top">Frames</a></li>
<li><a href="GeneInfo.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<div>
<ul class="subNavList">
<li>Summary:&nbsp;</li>
<li>Nested&nbsp;|&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_summary">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_summary">Method</a></li>
</ul>
<ul class="subNavList">
<li>Detail:&nbsp;</li>
<li>Field&nbsp;|&nbsp;</li>
<li><a href="#constructor_detail">Constr</a>&nbsp;|&nbsp;</li>
<li><a href="#method_detail">Method</a></li>
</ul>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
</body>
</html>
