<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_45) on Fri Jan 10 12:31:21 CET 2014 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Uses of Class org.jbiowhpersistence.datasets.protein.entities.Protein (jbiowh-persistence 6.0.1 API)</title>
<meta name="date" content="2014-01-10">
<link rel="stylesheet" type="text/css" href="../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Uses of Class org.jbiowhpersistence.datasets.protein.entities.Protein (jbiowh-persistence 6.0.1 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowhpersistence/datasets/protein/entities/class-use/Protein.html" target="_top">Frames</a></li>
<li><a href="Protein.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="header">
<h2 title="Uses of Class org.jbiowhpersistence.datasets.protein.entities.Protein" class="title">Uses of Class<br>org.jbiowhpersistence.datasets.protein.entities.Protein</h2>
</div>
<div class="classUseContainer">
<ul class="blockList">
<li class="blockList">
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing packages, and an explanation">
<caption><span>Packages that use <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Package</th>
<th class="colLast" scope="col">Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.domain.pfam.entities">org.jbiowhpersistence.datasets.domain.pfam.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.drug.drugbank.entities">org.jbiowhpersistence.datasets.drug.drugbank.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.gene.gene.entities">org.jbiowhpersistence.datasets.gene.gene.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.gene.genome.entities">org.jbiowhpersistence.datasets.gene.genome.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.ontology.entities">org.jbiowhpersistence.datasets.ontology.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.pathway.kegg.entities.enzyme">org.jbiowhpersistence.datasets.pathway.kegg.entities.enzyme</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway">org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.ppi.entities">org.jbiowhpersistence.datasets.ppi.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.protclust.entities">org.jbiowhpersistence.datasets.protclust.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.protein.controller">org.jbiowhpersistence.datasets.protein.controller</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.protein.entities">org.jbiowhpersistence.datasets.protein.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.protein.util">org.jbiowhpersistence.datasets.protein.util</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.protgroup.cog.entities">org.jbiowhpersistence.datasets.protgroup.cog.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.protgroup.pirsf.entities">org.jbiowhpersistence.datasets.protgroup.pirsf.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.utils.controller">org.jbiowhpersistence.utils.controller</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.utils.graph">org.jbiowhpersistence.utils.graph</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.utils.msms">org.jbiowhpersistence.utils.msms</a></td>
<td class="colLast">&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList">
<ul class="blockList">
<li class="blockList"><a name="org.jbiowhpersistence.datasets.domain.pfam.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/domain/pfam/entities/package-summary.html">org.jbiowhpersistence.datasets.domain.pfam.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/domain/pfam/entities/package-summary.html">org.jbiowhpersistence.datasets.domain.pfam.entities</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">PfamAbioWH.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/domain/pfam/entities/PfamAbioWH.html#getProteinInsignificant()">getProteinInsignificant</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">PfamAbioWH.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/domain/pfam/entities/PfamAbioWH.html#getProteinSignificant()">getProteinSignificant</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/datasets/domain/pfam/entities/package-summary.html">org.jbiowhpersistence.datasets.domain.pfam.entities</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">PfamAbioWH.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/domain/pfam/entities/PfamAbioWH.html#setProteinInsignificant(java.util.Set)">setProteinInsignificant</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;proteinInsignificant)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">PfamAbioWH.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/domain/pfam/entities/PfamAbioWH.html#setProteinSignificant(java.util.Set)">setProteinSignificant</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;proteinSignificant)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.drug.drugbank.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/package-summary.html">org.jbiowhpersistence.datasets.drug.drugbank.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/package-summary.html">org.jbiowhpersistence.datasets.drug.drugbank.entities</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">DrugBank.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/DrugBank.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">DrugBank.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/DrugBank.html#getProteinAsCarriers()">getProteinAsCarriers</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">DrugBank.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/DrugBank.html#getProteinAsEnzyme()">getProteinAsEnzyme</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">DrugBank.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/DrugBank.html#getProteinAsTransporters()">getProteinAsTransporters</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/package-summary.html">org.jbiowhpersistence.datasets.drug.drugbank.entities</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">DrugBank.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/DrugBank.html#setProtein(java.util.Set)">setProtein</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;protein)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">DrugBank.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/DrugBank.html#setProteinAsCarriers(java.util.Set)">setProteinAsCarriers</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;proteinAsCarriers)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">DrugBank.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/DrugBank.html#setProteinAsEnzyme(java.util.Set)">setProteinAsEnzyme</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;proteinAsEnzyme)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">DrugBank.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/drug/drugbank/entities/DrugBank.html#setProteinAsTransporters(java.util.Set)">setProteinAsTransporters</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;proteinAsTransporters)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.gene.gene.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.gene.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.gene.entities</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">GeneInfo.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.gene.entities</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">GeneInfo.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html#setProtein(java.util.Set)">setProtein</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.gene.genome.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.genome.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.genome.entities</a> that return <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></code></td>
<td class="colLast"><span class="strong">GenePTT.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/GenePTT.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.genome.entities</a> with parameters of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">GenePTT.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/GenePTT.html#setProtein(org.jbiowhpersistence.datasets.protein.entities.Protein)">setProtein</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.ontology.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/ontology/entities/package-summary.html">org.jbiowhpersistence.datasets.ontology.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/ontology/entities/package-summary.html">org.jbiowhpersistence.datasets.ontology.entities</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">Ontology.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/ontology/entities/Ontology.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/datasets/ontology/entities/package-summary.html">org.jbiowhpersistence.datasets.ontology.entities</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">Ontology.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/ontology/entities/Ontology.html#setProtein(java.util.Set)">setProtein</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.pathway.kegg.entities.enzyme">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/enzyme/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.enzyme</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/enzyme/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.enzyme</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">KEGGEnzyme.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/enzyme/KEGGEnzyme.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/enzyme/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.enzyme</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">KEGGEnzyme.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/enzyme/KEGGEnzyme.html#setProtein(java.util.Set)">setProtein</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">KEGGPathway.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/KEGGPathway.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">KEGGPathway.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/KEGGPathway.html#setProtein(java.util.Set)">setProtein</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.ppi.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/ppi/entities/package-summary.html">org.jbiowhpersistence.datasets.ppi.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/ppi/entities/package-summary.html">org.jbiowhpersistence.datasets.ppi.entities</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">MIFEntryInteraction.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/ppi/entities/MIFEntryInteraction.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/datasets/ppi/entities/package-summary.html">org.jbiowhpersistence.datasets.ppi.entities</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">MIFEntryInteraction.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/ppi/entities/MIFEntryInteraction.html#setProtein(java.util.Set)">setProtein</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.protclust.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/protclust/entities/package-summary.html">org.jbiowhpersistence.datasets.protclust.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protclust/entities/package-summary.html">org.jbiowhpersistence.datasets.protclust.entities</a> that return <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></code></td>
<td class="colLast"><span class="strong">UniRefMember.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protclust/entities/UniRefMember.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protclust/entities/package-summary.html">org.jbiowhpersistence.datasets.protclust.entities</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">UniRefEntry.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protclust/entities/UniRefEntry.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protclust/entities/package-summary.html">org.jbiowhpersistence.datasets.protclust.entities</a> with parameters of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">UniRefMember.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protclust/entities/UniRefMember.html#setProtein(org.jbiowhpersistence.datasets.protein.entities.Protein)">setProtein</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/datasets/protclust/entities/package-summary.html">org.jbiowhpersistence.datasets.protclust.entities</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">UniRefEntry.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protclust/entities/UniRefEntry.html#setProtein(java.util.Set)">setProtein</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.protein.controller">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/controller/package-summary.html">org.jbiowhpersistence.datasets.protein.controller</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/controller/package-summary.html">org.jbiowhpersistence.datasets.protein.controller</a> that return <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></code></td>
<td class="colLast"><span class="strong">ProteinJpaController.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/controller/ProteinJpaController.html#findProtein(java.lang.Long)">findProtein</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Long.html?is-external=true" title="class or interface in java.lang">Long</a>&nbsp;id)</code>
<div class="block">Find by primary key.</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/controller/package-summary.html">org.jbiowhpersistence.datasets.protein.controller</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">ProteinJpaController.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/controller/ProteinJpaController.html#findProteinEntities()">findProteinEntities</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">ProteinJpaController.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/controller/ProteinJpaController.html#findProteinEntities(int, int)">findProteinEntities</a></strong>(int&nbsp;maxResults,
                   int&nbsp;firstResult)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/controller/package-summary.html">org.jbiowhpersistence.datasets.protein.controller</a> with parameters of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">ProteinJpaController.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/controller/ProteinJpaController.html#create(org.jbiowhpersistence.datasets.protein.entities.Protein)">create</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;protein)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">ProteinJpaController.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/controller/ProteinJpaController.html#edit(org.jbiowhpersistence.datasets.protein.entities.Protein)">edit</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.protein.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/package-summary.html">org.jbiowhpersistence.datasets.protein.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/package-summary.html">org.jbiowhpersistence.datasets.protein.entities</a> that return <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></code></td>
<td class="colLast"><span class="strong">ProteinFeature.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/ProteinFeature.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></code></td>
<td class="colLast"><span class="strong">ProteinComment.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/ProteinComment.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></code></td>
<td class="colLast"><span class="strong">ProteinDBReference.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/ProteinDBReference.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></code></td>
<td class="colLast"><span class="strong">ProteinhasProteinKeyword.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/ProteinhasProteinKeyword.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/package-summary.html">org.jbiowhpersistence.datasets.protein.entities</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">ProteinKeyword.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/ProteinKeyword.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/package-summary.html">org.jbiowhpersistence.datasets.protein.entities</a> with parameters of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">ProteinFeature.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/ProteinFeature.html#setProtein(org.jbiowhpersistence.datasets.protein.entities.Protein)">setProtein</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;protein)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">ProteinComment.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/ProteinComment.html#setProtein(org.jbiowhpersistence.datasets.protein.entities.Protein)">setProtein</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;protein)</code>&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">ProteinDBReference.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/ProteinDBReference.html#setProtein(org.jbiowhpersistence.datasets.protein.entities.Protein)">setProtein</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;protein)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">ProteinhasProteinKeyword.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/ProteinhasProteinKeyword.html#setProtein(org.jbiowhpersistence.datasets.protein.entities.Protein)">setProtein</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/package-summary.html">org.jbiowhpersistence.datasets.protein.entities</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">ProteinKeyword.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/ProteinKeyword.html#setProtein(java.util.Set)">setProtein</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.protein.util">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/util/package-summary.html">org.jbiowhpersistence.datasets.protein.util</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/util/package-summary.html">org.jbiowhpersistence.datasets.protein.util</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">ProteinFromOther.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/util/ProteinFromOther.html#getProteinFromId(java.lang.String)">getProteinFromId</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>&nbsp;id)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">ProteinFromOther.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/util/ProteinFromOther.html#getProteinFromTaxonomy(org.jbiowhpersistence.datasets.taxonomy.entities.Taxonomy)">getProteinFromTaxonomy</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/taxonomy/entities/Taxonomy.html" title="class in org.jbiowhpersistence.datasets.taxonomy.entities">Taxonomy</a>&nbsp;taxonomy)</code>
<div class="block">Return the proteins that belong the Taxonomy</div>
</td>
</tr>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">ProteinFromOther.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/util/ProteinFromOther.html#getProteins()">getProteins</a></strong>()</code>
<div class="block">Return the list of founded proteins</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/util/package-summary.html">org.jbiowhpersistence.datasets.protein.util</a> with parameters of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><span class="strong">ProteinProteinGraph.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/util/ProteinProteinGraph.html#populate(org.jbiowhpersistence.datasets.protein.entities.Protein, int)">populate</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from Protein entity.</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/datasets/protein/util/package-summary.html">org.jbiowhpersistence.datasets.protein.util</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">ProteinUtils.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/util/ProteinUtils.html#printProteinName(java.io.PrintStream, java.util.List)">printProteinName</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/io/PrintStream.html?is-external=true" title="class or interface in java.io">PrintStream</a>&nbsp;out,
                <a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;proteins)</code>
<div class="block">Print the protein ID (name) on the PrintStream file</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">ProteinUtils.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protein/util/ProteinUtils.html#printProteinName(java.io.PrintWriter, java.util.List)">printProteinName</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/io/PrintWriter.html?is-external=true" title="class or interface in java.io">PrintWriter</a>&nbsp;writer,
                <a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;proteins)</code>
<div class="block">Print the protein ID (name) on the write file</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.protgroup.cog.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/protgroup/cog/entities/package-summary.html">org.jbiowhpersistence.datasets.protgroup.cog.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protgroup/cog/entities/package-summary.html">org.jbiowhpersistence.datasets.protgroup.cog.entities</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">COGOrthologousGroup.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protgroup/cog/entities/COGOrthologousGroup.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/datasets/protgroup/cog/entities/package-summary.html">org.jbiowhpersistence.datasets.protgroup.cog.entities</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">COGOrthologousGroup.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protgroup/cog/entities/COGOrthologousGroup.html#setProtein(java.util.Set)">setProtein</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.protgroup.pirsf.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/datasets/protgroup/pirsf/entities/package-summary.html">org.jbiowhpersistence.datasets.protgroup.pirsf.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protgroup/pirsf/entities/package-summary.html">org.jbiowhpersistence.datasets.protgroup.pirsf.entities</a> that return <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></code></td>
<td class="colLast"><span class="strong">PirsfhasProtein.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protgroup/pirsf/entities/PirsfhasProtein.html#getProtein()">getProtein</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/datasets/protgroup/pirsf/entities/package-summary.html">org.jbiowhpersistence.datasets.protgroup.pirsf.entities</a> with parameters of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">PirsfhasProtein.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/datasets/protgroup/pirsf/entities/PirsfhasProtein.html#setProtein(org.jbiowhpersistence.datasets.protein.entities.Protein)">setProtein</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;protein)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.utils.controller">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/utils/controller/package-summary.html">org.jbiowhpersistence.utils.controller</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/utils/controller/package-summary.html">org.jbiowhpersistence.utils.controller</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>protected <a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">AbstractJpaController.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/utils/controller/AbstractJpaController.html#createProtein(javax.persistence.EntityManagerFactory, javax.persistence.EntityManager, java.util.Set)">createProtein</a></strong>(javax.persistence.EntityManagerFactory&nbsp;emf,
             javax.persistence.EntityManager&nbsp;em,
             <a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;objectList)</code>
<div class="block">Return the Protein list the be persisted</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/utils/controller/package-summary.html">org.jbiowhpersistence.utils.controller</a> with parameters of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>protected <a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protclust/entities/UniRefEntry.html" title="class in org.jbiowhpersistence.datasets.protclust.entities">UniRefEntry</a>&gt;</code></td>
<td class="colLast"><span class="strong">AbstractJpaController.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/utils/controller/AbstractJpaController.html#createUniRefEntry(javax.persistence.EntityManagerFactory, javax.persistence.EntityManager, java.util.Set, org.jbiowhpersistence.datasets.protein.entities.Protein)">createUniRefEntry</a></strong>(javax.persistence.EntityManagerFactory&nbsp;emf,
                 javax.persistence.EntityManager&nbsp;em,
                 <a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protclust/entities/UniRefEntry.html" title="class in org.jbiowhpersistence.datasets.protclust.entities">UniRefEntry</a>&gt;&nbsp;objectList,
                 <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;protein)</code>
<div class="block">Return the UniRefEntry list the be persisted</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../org/jbiowhpersistence/utils/controller/package-summary.html">org.jbiowhpersistence.utils.controller</a> with type arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>protected <a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;</code></td>
<td class="colLast"><span class="strong">AbstractJpaController.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/utils/controller/AbstractJpaController.html#createProtein(javax.persistence.EntityManagerFactory, javax.persistence.EntityManager, java.util.Set)">createProtein</a></strong>(javax.persistence.EntityManagerFactory&nbsp;emf,
             javax.persistence.EntityManager&nbsp;em,
             <a href="http://docs.oracle.com/javase/6/docs/api/java/util/Set.html?is-external=true" title="class or interface in java.util">Set</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&nbsp;objectList)</code>
<div class="block">Return the Protein list the be persisted</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.utils.graph">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/utils/graph/package-summary.html">org.jbiowhpersistence.utils.graph</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/utils/graph/package-summary.html">org.jbiowhpersistence.utils.graph</a> with parameters of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><span class="strong">JBioWHGraph.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.protein.entities.Protein, int)">populate</a></strong>(<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from Protein entity</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.utils.msms">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a> in <a href="../../../../../../org/jbiowhpersistence/utils/msms/package-summary.html">org.jbiowhpersistence.utils.msms</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../org/jbiowhpersistence/utils/msms/package-summary.html">org.jbiowhpersistence.utils.msms</a> that return types with arguments of type <a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/LinkedList.html?is-external=true" title="class or interface in java.util">LinkedList</a>&lt;<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Map.Entry.html?is-external=true" title="class or interface in java.util">Map.Entry</a>&lt;<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>,<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&gt;&gt;</code></td>
<td class="colLast"><span class="strong">MSData.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/utils/msms/MSData.html#getProteinList()">getProteinList</a></strong>()</code>
<div class="block">Get the protein list extracted from the MS XML result file</div>
</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/LinkedList.html?is-external=true" title="class or interface in java.util">LinkedList</a>&lt;<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Map.Entry.html?is-external=true" title="class or interface in java.util">Map.Entry</a>&lt;<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/String.html?is-external=true" title="class or interface in java.lang">String</a>,<a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Protein</a>&gt;&gt;&gt;</code></td>
<td class="colLast"><span class="strong">MSFactory.</span><code><strong><a href="../../../../../../org/jbiowhpersistence/utils/msms/MSFactory.html#getProteinList()">getProteinList</a></strong>()</code>
<div class="block">Get the protein list extracted from the MS XML result file</div>
</td>
</tr>
</tbody>
</table>
</li>
</ul>
</li>
</ul>
</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html" title="class in org.jbiowhpersistence.datasets.protein.entities">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../index.html?org/jbiowhpersistence/datasets/protein/entities/class-use/Protein.html" target="_top">Frames</a></li>
<li><a href="Protein.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2014. All rights reserved.</small></p>
</body>
</html>
