<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
<!-- NewPage -->
<html lang="en">
<head>
<!-- Generated by javadoc (version 1.7.0_45) on Fri Jan 10 12:31:21 CET 2014 -->
<meta http-equiv="Content-Type" content="text/html" charset="UTF-8">
<title>Uses of Class org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo (jbiowh-persistence 6.0.1 API)</title>
<meta name="date" content="2014-01-10">
<link rel="stylesheet" type="text/css" href="../../../../../../../stylesheet.css" title="Style">
</head>
<body>
<script type="text/javascript"><!--
    if (location.href.indexOf('is-external=true') == -1) {
        parent.document.title="Uses of Class org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo (jbiowh-persistence 6.0.1 API)";
    }
//-->
</script>
<noscript>
<div>JavaScript is disabled on your browser.</div>
</noscript>
<!-- ========= START OF TOP NAVBAR ======= -->
<div class="topNav"><a name="navbar_top">
<!--   -->
</a><a href="#skip-navbar_top" title="Skip navigation links"></a><a name="navbar_top_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../../index.html?org/jbiowhpersistence/datasets/gene/gene/entities/class-use/GeneInfo.html" target="_top">Frames</a></li>
<li><a href="GeneInfo.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_top">
<li><a href="../../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_top");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_top">
<!--   -->
</a></div>
<!-- ========= END OF TOP NAVBAR ========= -->
<div class="header">
<h2 title="Uses of Class org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo" class="title">Uses of Class<br>org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo</h2>
</div>
<div class="classUseContainer">
<ul class="blockList">
<li class="blockList">
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing packages, and an explanation">
<caption><span>Packages that use <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Package</th>
<th class="colLast" scope="col">Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.disease.omim.entities">org.jbiowhpersistence.datasets.disease.omim.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.gene.gene.controller">org.jbiowhpersistence.datasets.gene.gene.controller</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.gene.genebank.entities">org.jbiowhpersistence.datasets.gene.genebank.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.gene.genome.entities">org.jbiowhpersistence.datasets.gene.genome.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.gene.genome.util">org.jbiowhpersistence.datasets.gene.genome.util</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.ontology.entities">org.jbiowhpersistence.datasets.ontology.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.pathway.kegg.entities.gene">org.jbiowhpersistence.datasets.pathway.kegg.entities.gene</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway">org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.protein.entities">org.jbiowhpersistence.datasets.protein.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.datasets.protgroup.cog.entities">org.jbiowhpersistence.datasets.protgroup.cog.entities</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="altColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.utils.controller">org.jbiowhpersistence.utils.controller</a></td>
<td class="colLast">&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><a href="#org.jbiowhpersistence.utils.graph">org.jbiowhpersistence.utils.graph</a></td>
<td class="colLast">&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList">
<ul class="blockList">
<li class="blockList"><a name="org.jbiowhpersistence.datasets.disease.omim.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/datasets/disease/omim/entities/package-summary.html">org.jbiowhpersistence.datasets.disease.omim.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/disease/omim/entities/package-summary.html">org.jbiowhpersistence.datasets.disease.omim.entities</a> that return types with arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><span class="strong">OMIM.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/disease/omim/entities/OMIM.html#getGeneInfo()">getGeneInfo</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../../org/jbiowhpersistence/datasets/disease/omim/entities/package-summary.html">org.jbiowhpersistence.datasets.disease.omim.entities</a> with type arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">OMIM.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/disease/omim/entities/OMIM.html#setGeneInfo(java.util.Collection)">setGeneInfo</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;&nbsp;geneInfo)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.gene.gene.controller">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/controller/package-summary.html">org.jbiowhpersistence.datasets.gene.gene.controller</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/controller/package-summary.html">org.jbiowhpersistence.datasets.gene.gene.controller</a> that return <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></code></td>
<td class="colLast"><span class="strong">GeneInfoJpaController.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/controller/GeneInfoJpaController.html#findGeneInfo(java.lang.Long)">findGeneInfo</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/lang/Long.html?is-external=true" title="class or interface in java.lang">Long</a>&nbsp;id)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/controller/package-summary.html">org.jbiowhpersistence.datasets.gene.gene.controller</a> that return types with arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><span class="strong">GeneInfoJpaController.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/controller/GeneInfoJpaController.html#findGeneInfoEntities()">findGeneInfoEntities</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/List.html?is-external=true" title="class or interface in java.util">List</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><span class="strong">GeneInfoJpaController.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/controller/GeneInfoJpaController.html#findGeneInfoEntities(int, int)">findGeneInfoEntities</a></strong>(int&nbsp;maxResults,
                    int&nbsp;firstResult)</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/controller/package-summary.html">org.jbiowhpersistence.datasets.gene.gene.controller</a> with parameters of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">GeneInfoJpaController.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/controller/GeneInfoJpaController.html#create(org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo)">create</a></strong>(<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&nbsp;geneInfo)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">GeneInfoJpaController.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/controller/GeneInfoJpaController.html#edit(org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo)">edit</a></strong>(<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&nbsp;geneInfo)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.gene.genebank.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genebank/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.genebank.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genebank/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.genebank.entities</a> that return types with arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><span class="strong">GeneBankCDS.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genebank/entities/GeneBankCDS.html#getGeneInfo()">getGeneInfo</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genebank/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.genebank.entities</a> with type arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">GeneBankCDS.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genebank/entities/GeneBankCDS.html#setGeneInfo(java.util.Collection)">setGeneInfo</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;&nbsp;geneInfo)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.gene.genome.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.genome.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.genome.entities</a> that return <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></code></td>
<td class="colLast"><span class="strong">GenePTT.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/GenePTT.html#getGeneInfo()">getGeneInfo</a></strong>()</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></code></td>
<td class="colLast"><span class="strong">GeneRNT.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/GeneRNT.html#getGeneInfo()">getGeneInfo</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/package-summary.html">org.jbiowhpersistence.datasets.gene.genome.entities</a> with parameters of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">GenePTT.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/GenePTT.html#setGeneInfo(org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo)">setGeneInfo</a></strong>(<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&nbsp;geneInfo)</code>&nbsp;</td>
</tr>
<tr class="rowColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">GeneRNT.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genome/entities/GeneRNT.html#setGeneInfo(org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo)">setGeneInfo</a></strong>(<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&nbsp;geneInfo)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.gene.genome.util">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genome/util/package-summary.html">org.jbiowhpersistence.datasets.gene.genome.util</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genome/util/package-summary.html">org.jbiowhpersistence.datasets.gene.genome.util</a> with parameters of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">Chromosome.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/genome/util/Chromosome.html#createChromosomeFromGeneInfo(org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo)">createChromosomeFromGeneInfo</a></strong>(<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&nbsp;gene)</code>
<div class="block">Create a Chromosome from one of its genes</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.ontology.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/datasets/ontology/entities/package-summary.html">org.jbiowhpersistence.datasets.ontology.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/ontology/entities/package-summary.html">org.jbiowhpersistence.datasets.ontology.entities</a> that return types with arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><span class="strong">Ontology.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/ontology/entities/Ontology.html#getGeneInfo()">getGeneInfo</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../../org/jbiowhpersistence/datasets/ontology/entities/package-summary.html">org.jbiowhpersistence.datasets.ontology.entities</a> with type arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">Ontology.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/ontology/entities/Ontology.html#setGeneInfo(java.util.Collection)">setGeneInfo</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;&nbsp;geneInfo)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.pathway.kegg.entities.gene">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/gene/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.gene</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/gene/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.gene</a> that return types with arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><span class="strong">KEGGGene.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/gene/KEGGGene.html#getGeneInfos()">getGeneInfos</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/gene/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.gene</a> with type arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">KEGGGene.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/gene/KEGGGene.html#setGeneInfos(java.util.Collection)">setGeneInfos</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;&nbsp;geneInfos)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway</a> that return types with arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><span class="strong">KEGGPathway.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/KEGGPathway.html#getGeneInfo()">getGeneInfo</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/package-summary.html">org.jbiowhpersistence.datasets.pathway.kegg.entities.pathway</a> with type arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">KEGGPathway.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/pathway/kegg/entities/pathway/KEGGPathway.html#setGeneInfo(java.util.Collection)">setGeneInfo</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;&nbsp;geneInfos)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.protein.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/datasets/protein/entities/package-summary.html">org.jbiowhpersistence.datasets.protein.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/protein/entities/package-summary.html">org.jbiowhpersistence.datasets.protein.entities</a> that return types with arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><span class="strong">Protein.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html#getGeneInfo()">getGeneInfo</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../../org/jbiowhpersistence/datasets/protein/entities/package-summary.html">org.jbiowhpersistence.datasets.protein.entities</a> with type arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">Protein.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/protein/entities/Protein.html#setGeneInfo(java.util.Collection)">setGeneInfo</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;&nbsp;geneInfo)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.datasets.protgroup.cog.entities">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/datasets/protgroup/cog/entities/package-summary.html">org.jbiowhpersistence.datasets.protgroup.cog.entities</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/datasets/protgroup/cog/entities/package-summary.html">org.jbiowhpersistence.datasets.protgroup.cog.entities</a> that return types with arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code><a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><span class="strong">COGOrthologousGroup.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/protgroup/cog/entities/COGOrthologousGroup.html#getGeneInfo()">getGeneInfo</a></strong>()</code>&nbsp;</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../../org/jbiowhpersistence/datasets/protgroup/cog/entities/package-summary.html">org.jbiowhpersistence.datasets.protgroup.cog.entities</a> with type arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>void</code></td>
<td class="colLast"><span class="strong">COGOrthologousGroup.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/datasets/protgroup/cog/entities/COGOrthologousGroup.html#setGeneInfo(java.util.Collection)">setGeneInfo</a></strong>(<a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;&nbsp;geneInfo)</code>&nbsp;</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.utils.controller">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/utils/controller/package-summary.html">org.jbiowhpersistence.utils.controller</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/utils/controller/package-summary.html">org.jbiowhpersistence.utils.controller</a> that return types with arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>protected <a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><span class="strong">AbstractJpaController.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/utils/controller/AbstractJpaController.html#createGeneInfo(javax.persistence.EntityManagerFactory, javax.persistence.EntityManager, java.util.Collection)">createGeneInfo</a></strong>(javax.persistence.EntityManagerFactory&nbsp;emf,
              javax.persistence.EntityManager&nbsp;em,
              <a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;&nbsp;objectList)</code>
<div class="block">Return the GeneInfo list the be persisted</div>
</td>
</tr>
</tbody>
</table>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Method parameters in <a href="../../../../../../../org/jbiowhpersistence/utils/controller/package-summary.html">org.jbiowhpersistence.utils.controller</a> with type arguments of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>protected <a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;</code></td>
<td class="colLast"><span class="strong">AbstractJpaController.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/utils/controller/AbstractJpaController.html#createGeneInfo(javax.persistence.EntityManagerFactory, javax.persistence.EntityManager, java.util.Collection)">createGeneInfo</a></strong>(javax.persistence.EntityManagerFactory&nbsp;emf,
              javax.persistence.EntityManager&nbsp;em,
              <a href="http://docs.oracle.com/javase/6/docs/api/java/util/Collection.html?is-external=true" title="class or interface in java.util">Collection</a>&lt;<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&gt;&nbsp;objectList)</code>
<div class="block">Return the GeneInfo list the be persisted</div>
</td>
</tr>
</tbody>
</table>
</li>
<li class="blockList"><a name="org.jbiowhpersistence.utils.graph">
<!--   -->
</a>
<h3>Uses of <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a> in <a href="../../../../../../../org/jbiowhpersistence/utils/graph/package-summary.html">org.jbiowhpersistence.utils.graph</a></h3>
<table border="0" cellpadding="3" cellspacing="0" summary="Use table, listing methods, and an explanation">
<caption><span>Methods in <a href="../../../../../../../org/jbiowhpersistence/utils/graph/package-summary.html">org.jbiowhpersistence.utils.graph</a> with parameters of type <a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a></span><span class="tabEnd">&nbsp;</span></caption>
<tr>
<th class="colFirst" scope="col">Modifier and Type</th>
<th class="colLast" scope="col">Method and Description</th>
</tr>
<tbody>
<tr class="altColor">
<td class="colFirst"><code>protected void</code></td>
<td class="colLast"><span class="strong">JBioWHGraph.</span><code><strong><a href="../../../../../../../org/jbiowhpersistence/utils/graph/JBioWHGraph.html#populate(org.jbiowhpersistence.datasets.gene.gene.entities.GeneInfo, int)">populate</a></strong>(<a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">GeneInfo</a>&nbsp;o,
        int&nbsp;varDepth)</code>
<div class="block">The recursive function to populate the graph from GeneInfo entity</div>
</td>
</tr>
</tbody>
</table>
</li>
</ul>
</li>
</ul>
</div>
<!-- ======= START OF BOTTOM NAVBAR ====== -->
<div class="bottomNav"><a name="navbar_bottom">
<!--   -->
</a><a href="#skip-navbar_bottom" title="Skip navigation links"></a><a name="navbar_bottom_firstrow">
<!--   -->
</a>
<ul class="navList" title="Navigation">
<li><a href="../../../../../../../overview-summary.html">Overview</a></li>
<li><a href="../package-summary.html">Package</a></li>
<li><a href="../../../../../../../org/jbiowhpersistence/datasets/gene/gene/entities/GeneInfo.html" title="class in org.jbiowhpersistence.datasets.gene.gene.entities">Class</a></li>
<li class="navBarCell1Rev">Use</li>
<li><a href="../package-tree.html">Tree</a></li>
<li><a href="../../../../../../../deprecated-list.html">Deprecated</a></li>
<li><a href="../../../../../../../index-all.html">Index</a></li>
<li><a href="../../../../../../../help-doc.html">Help</a></li>
</ul>
</div>
<div class="subNav">
<ul class="navList">
<li>Prev</li>
<li>Next</li>
</ul>
<ul class="navList">
<li><a href="../../../../../../../index.html?org/jbiowhpersistence/datasets/gene/gene/entities/class-use/GeneInfo.html" target="_top">Frames</a></li>
<li><a href="GeneInfo.html" target="_top">No Frames</a></li>
</ul>
<ul class="navList" id="allclasses_navbar_bottom">
<li><a href="../../../../../../../allclasses-noframe.html">All Classes</a></li>
</ul>
<div>
<script type="text/javascript"><!--
  allClassesLink = document.getElementById("allclasses_navbar_bottom");
  if(window==top) {
    allClassesLink.style.display = "block";
  }
  else {
    allClassesLink.style.display = "none";
  }
  //-->
</script>
</div>
<a name="skip-navbar_bottom">
<!--   -->
</a></div>
<!-- ======== END OF BOTTOM NAVBAR ======= -->
<p class="legalCopy"><small>Copyright &#169; 2014. All rights reserved.</small></p>
</body>
</html>
